<!DOCTYPE html>
<html>
	<head>
		<title>R-GenomicAlignments - Details</title>
		<link rel="stylesheet" href="../../assets/css/bootstrap.min.css">
		<link rel="stylesheet" href="../../assets/css/main.css">
		<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
	</head>
	<body>
		<div class="container">
			<div>
				<div class="row">
					<a href="/" class="col-md-6">
						<div>
							<img src="../../../assets/images/logo.png" style="margin-top: -30px;">
						</div>
					</a>
					<div class="col-md-6">
						
					</div>
				</div>
			</div>

			<hr />

			<h1>
				R-GenomicAlignments
				<small class="text-muted"> in Fedora 33</small>
			</h1>
			<p>
			This page displays detailed information for a specific version of R-GenomicAlignments. You might
			want go back to <a href=".">the main page of R-GenomicAlignments</a>.
			</p>

			<h2>Changelog</h2>
			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Date</th>
						<th scope="col">Author</th>
						<th scope="col">Change</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>2020-07-27</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.22.1-3</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2020-07-04</td>
					<td>José Abílio Matos <jamatos at fc dot up dot pt> - 1.22.1-2</td>
					<td>- rebuild for R 4</td>
				</tr>
				
				<tr scope="row">
					<td>2020-02-01</td>
					<td>Tom Callaway <spot at fedoraproject dot org> - 1.22.1-1</td>
					<td>- update to 1.22.1</td>
				</tr>
				
				<tr scope="row">
					<td>2020-01-28</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.22.0-3</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-11-21</td>
					<td>Elliott Sales de Andrade <quantum dot analyst at gmail dot com> - 1.22.0-2</td>
					<td>- Exclude Suggests for unavailable packages</td>
				</tr>
				
				<tr scope="row">
					<td>2019-11-06</td>
					<td>Tom Callaway <spot at fedoraproject dot org> - 1.22.0-1</td>
					<td>- update to 1.22.0</td>
				</tr>
				
				<tr scope="row">
					<td>2019-08-12</td>
					<td>Elliott Sales de Andrade <quantum dot analyst at gmail dot com> - 1.18.1-3</td>
					<td>- Remove explicit dependencies provided by automatic dependency generator</td>
				</tr>
				
				<tr scope="row">
					<td>2019-07-24</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.18.1-2</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_31_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-02-06</td>
					<td>Tom Callaway <spot at fedoraproject dot org> - 1.18.1-1</td>
					<td>- update to 1.18.1</td>
				</tr>
				
				<tr scope="row">
					<td>2019-01-31</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 1.16.0-3</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_30_Mass_Rebuild</td>
				</tr>
				
				<tbody>
				</tbody>
			</table>

			<h2>Files</h2>

			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Type</th>
						<th scope="col">Path</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/GenomicAlignmentsIntroduction.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/GenomicAlignmentsIntroduction.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/GenomicAlignmentsIntroduction.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/OverlapEncodings.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/OverlapEncodings.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/OverlapEncodings.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/WorkingWithAlignedNucleotides.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/WorkingWithAlignedNucleotides.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/WorkingWithAlignedNucleotides.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/index.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/summarizeOverlaps.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/summarizeOverlaps.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc/summarizeOverlaps.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help/AnIndex</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help/GenomicAlignments.rdb</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help/GenomicAlignments.rdx</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help/aliases.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help/paths.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/CITATION</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/DESCRIPTION</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/INDEX</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/NAMESPACE</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/NEWS</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/R</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/doc</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/extdata</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/help</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/libs</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/.build-id/a7</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/Rd.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/features.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/hsearch.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/links.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/nsInfo.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/package.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/Meta/vignette.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/00Index.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/GAlignmentPairs-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/GAlignments-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/GAlignmentsList-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/GappedReads-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/OverlapEncodings-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/R.css</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/cigar-utils.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/coordinate-mapping-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/coverage-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/encodeOverlaps-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/findCompatibleOverlaps-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/findMateAlignment.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/findOverlaps-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/findSpliceOverlaps-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/intra-range-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/junctions-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/pileLettersAt.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/readGAlignments.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/sequenceLayer.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/setops-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/stackStringsFromGAlignments.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/html/summarizeOverlaps-methods.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/R/GenomicAlignments</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/R/GenomicAlignments.rdb</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/R/GenomicAlignments.rdx</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/GenomicAlignments</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/.build-id</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/.build-id/a7/bcf0530ee43b4367f76eb20fb8b46f7fc41823</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/libs/GenomicAlignments.so</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_GAlignments-class.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_GAlignmentsList-class.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_cigar-utils.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_coordinate-mapping-methods.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_findSpliceOverlaps-methods.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_intra-range-methods.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_readGAlignmentPairs.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_readGAlignments.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_readGAlignmentsList.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/unitTests/test_summarizeOverlaps-methods.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/extdata/sm_treated1.bam</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/GenomicAlignments/extdata/sm_untreated1.bam</td>
				</tr>
				
				<tbody>
		</div>
	</body>
</html>