<!DOCTYPE html>

<title>R-GenomicAlignments-1.28.0-3.fc36 - Fedora Packages</title>
<link rel="stylesheet" href="../../../assets/css/bootstrap.min.css">
<link rel="stylesheet" href="../../../assets/css/main.css">
<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta charset="UTF-8">
<meta name="description" content="View R-GenomicAlignments-1.28.0-3.fc36 in Fedora 36. R-GenomicAlignments: Representation and manipulation of short genomic alignments">

<div class="container">
  <div class="row">
    <a href="../../../" class="col-md-6 d-flex align-items-center justify-content-center justify-content-md-start p-1">
      <img alt="Fedora Packages" src="../../../assets/images/logo.png">
    </a>
    <div class="col-md-6 d-flex align-items-center">
    </div>
  </div>

  <hr />

<h1>
	R-GenomicAlignments-1.28.0-3.fc36<small class="text-muted"> in Fedora 36</small>
</h1>
<p>
	<a href=".">&crarr; Return to the main page of R-GenomicAlignments</a><br>
	<a href="https://koji.fedoraproject.org/koji/search?match=exact&type=build&terms=R-GenomicAlignments-1.28.0-3.fc36">View build</a><br>
	<a href="https://bodhi.fedoraproject.org/updates/?search=R-GenomicAlignments-1.28.0-3.fc36">Search for updates</a>
</p>

<p>
	<b>Package Info <span class="text-muted">(Data from x86_64 build)</span></b><br>
	<a href="#changelog">&#129047; Changelog</a><br>
	<a href="#dependencies">&#129047; Dependencies</a><br>
	<a href="#provides">&#129047; Provides</a><br>
	<a href="#files">&#129047; Files</a><br>
</p>

<h2 id="changelog">Changelog</h2>
<div class="table-responsive">
	<table class="table table-striped table-borderless ">
		<thead>
			<tr>
				<th scope="col">Date</th>
				<th scope="col">Author</th>
				<th scope="col">Change</th>
			</tr>
		</thead>
		<tr>
			<td>2022-01-19</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.28.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_36_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-07-21</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.28.0-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_35_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-06-17</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.28.0-1</td>
			<td>- update to 1.28.0
- Rebuilt for R 4.1.0</td>
		</tr>
		<tr>
			<td>2021-02-04</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.26.0-1</td>
			<td>- update to 1.26.0</td>
		</tr>
		<tr>
			<td>2021-01-25</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.22.1-4</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2020-07-27</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.22.1-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2020-07-04</td>
			<td>José Abílio Matos &lt;jamatos at fc dot up dot pt&gt; - 1.22.1-2</td>
			<td>- rebuild for R 4</td>
		</tr>
		<tr>
			<td>2020-02-01</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.22.1-1</td>
			<td>- update to 1.22.1</td>
		</tr>
		<tr>
			<td>2020-01-28</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.22.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
		</tr>
		<tbody>
		</tbody>
	</table>
</div>

<div class="row">
	<div class="col">
		<h2 id="dependencies">Dependencies</h2>
		<ul>
			<li>
			<a href="/pkgs/R-BiocGenerics/R-BiocGenerics/fedora-36.html">
			R-BiocGenerics</a> &gt;= 0.37.0

			</li>
			<li>
			<a href="/pkgs/R-BiocParallel/R-BiocParallel/fedora-36.html">
			R-BiocParallel</a>

			</li>
			<li>
			<a href="/pkgs/R-Biostrings/R-Biostrings/fedora-36.html">
			R-Biostrings</a> &gt;= 2.55.7

			</li>
			<li>
			<a href="/pkgs/R-GenomeInfoDb/R-GenomeInfoDb/fedora-36.html">
			R-GenomeInfoDb</a> &gt;= 1.13.1

			</li>
			<li>
			<a href="/pkgs/R-GenomicRanges/R-GenomicRanges/fedora-36.html">
			R-GenomicRanges</a> &gt;= 1.41.5

			</li>
			<li>
			<a href="/pkgs/R-IRanges/R-IRanges/fedora-36.html">
			R-IRanges</a> &gt;= 2.23.9

			</li>
			<li>
			<a href="/pkgs/R-Rsamtools/R-Rsamtools/fedora-36.html">
			R-Rsamtools</a> &gt;= 1.31.2

			</li>
			<li>
			<a href="/pkgs/R-S4Vectors/R-S4Vectors/fedora-36.html">
			R-S4Vectors</a> &gt;= 0.27.12

			</li>
			<li>
			<a href="/pkgs/R-SummarizedExperiment/R-SummarizedExperiment/fedora-36.html">
			R-SummarizedExperiment</a> &gt;= 1.9.13

			</li>
			<li>
			<a href="/pkgs/R/R-core/fedora-36.html">
			R-core</a> = 4.1

			</li>
			<li>
			<a href="/pkgs/glibc/glibc/fedora-36.html">
			glibc</a>

			</li>
		</ul>
	</div>
	<div class="col">
		<h2 id="provides">Provides</h2>
		<ul>
			<li>R(GenomicAlignments)</li>
			<li>R-GenomicAlignments</li>
			<li>R-GenomicAlignments(x86-64)</li>
		</ul>
	</div>
</div>

<h2 id="files">Files</h2>

<div class="tree">
	<ul>
	<li>usr/<ul>
	<li>lib/<ul>
	<li>.build-id/<ul>
	<li>37/<ul>
	<li>bcdcd71196fdd5aa07d16d9dda5b3ed9c9a1c6</li>
	</ul></li>
	</ul></li>
	</ul></li>
	<li>lib64/<ul>
	<li>R/<ul>
	<li>library/<ul>
	<li>GenomicAlignments/<ul>
	<li>CITATION</li>
	<li>DESCRIPTION</li>
	<li>INDEX</li>
	<li>Meta/<ul>
	<li>Rd.rds</li>
	<li>features.rds</li>
	<li>hsearch.rds</li>
	<li>links.rds</li>
	<li>nsInfo.rds</li>
	<li>package.rds</li>
	<li>vignette.rds</li>
	</ul></li>
	<li>NAMESPACE</li>
	<li>NEWS</li>
	<li>R/<ul>
	<li>GenomicAlignments</li>
	<li>GenomicAlignments.rdb</li>
	<li>GenomicAlignments.rdx</li>
	</ul></li>
	<li>doc/<ul>
	<li>GenomicAlignmentsIntroduction.R</li>
	<li>GenomicAlignmentsIntroduction.Rnw</li>
	<li>GenomicAlignmentsIntroduction.pdf</li>
	<li>OverlapEncodings.R</li>
	<li>OverlapEncodings.Rnw</li>
	<li>OverlapEncodings.pdf</li>
	<li>WorkingWithAlignedNucleotides.R</li>
	<li>WorkingWithAlignedNucleotides.Rnw</li>
	<li>WorkingWithAlignedNucleotides.pdf</li>
	<li>index.html</li>
	<li>summarizeOverlaps.R</li>
	<li>summarizeOverlaps.Rnw</li>
	<li>summarizeOverlaps.pdf</li>
	</ul></li>
	<li>extdata/<ul>
	<li>sm_treated1.bam</li>
	<li>sm_untreated1.bam</li>
	</ul></li>
	<li>help/<ul>
	<li>+5B+5B+2CGAlignmentPairs+2CANY+2CANY-method.html</li>
	<li>AnIndex</li>
	<li>CIGAR_OPS.html</li>
	<li>GAlignmentPairs-class.html</li>
	<li>GAlignmentPairs.html</li>
	<li>GAlignments-class.html</li>
	<li>GAlignments.html</li>
	<li>GAlignmentsList-class.html</li>
	<li>GAlignmentsList.html</li>
	<li>GappedReads-class.html</li>
	<li>GappedReads.html</li>
	<li>GenomicAlignments.rdb</li>
	<li>GenomicAlignments.rdx</li>
	<li>IntersectionNotEmpty.html</li>
	<li>IntersectionStrict.html</li>
	<li>Lencoding.html</li>
	<li>Lnjunc.html</li>
	<li>Loffset+2COverlapEncodings-method.html</li>
	<li>Loffset.html</li>
	<li>NATURAL_INTRON_MOTIFS.html</li>
	<li>OverlapEncodings-class.html</li>
	<li>OverlapEncodings.html</li>
	<li>Rencoding.html</li>
	<li>Rnjunc.html</li>
	<li>Roffset+2COverlapEncodings-method.html</li>
	<li>Roffset.html</li>
	<li>Union.html</li>
	<li>aliases.rds</li>
	<li>alphabetFrequencyFromBam.html</li>
	<li>as.data.frame+2CGAlignmentPairs-method.html</li>
	<li>as.data.frame+2CGAlignments-method.html</li>
	<li>as.data.frame+2COverlapEncodings-method.html</li>
	<li>as.data.frame.OverlapEncodings.html</li>
	<li>bindROWS+2CGAlignmentPairs-method.html</li>
	<li>bindROWS+2CGAlignments-method.html</li>
	<li>c+2CGappedReads-method.html</li>
	<li>cigar+2CGAlignments-method.html</li>
	<li>cigar+2CGAlignmentsList-method.html</li>
	<li>cigar-utils.html</li>
	<li>cigar.html</li>
	<li>cigarNarrow.html</li>
	<li>cigarOpTable.html</li>
	<li>cigarQNarrow.html</li>
	<li>cigarRangesAlongPairwiseSpace.html</li>
	<li>cigarRangesAlongQuerySpace.html</li>
	<li>cigarRangesAlongReferenceSpace.html</li>
	<li>cigarToRleList.html</li>
	<li>cigarWidthAlongPairwiseSpace.html</li>
	<li>cigarWidthAlongQuerySpace.html</li>
	<li>cigarWidthAlongReferenceSpace.html</li>
	<li>class+3AGAlignmentPairs.html</li>
	<li>class+3AGAlignments.html</li>
	<li>class+3AGAlignmentsList.html</li>
	<li>class+3AGappedReads.html</li>
	<li>class+3AOverlapEncodings.html</li>
	<li>coerce+2CGAlignmentPairs+2CDataFrame-method.html</li>
	<li>coerce+2CGAlignmentPairs+2CGAlignments-method.html</li>
	<li>coerce+2CGAlignmentPairs+2CGAlignmentsList-method.html</li>
	<li>coerce+2CGAlignmentPairs+2CGRanges-method.html</li>
	<li>coerce+2CGAlignmentPairs+2CGRangesList-method.html</li>
	<li>coerce+2CGAlignmentPairs+2CIntegerRanges-method.html</li>
	<li>coerce+2CGAlignments+2CDataFrame-method.html</li>
	<li>coerce+2CGAlignments+2CGRanges-method.html</li>
	<li>coerce+2CGAlignments+2CGRangesList-method.html</li>
	<li>coerce+2CGAlignments+2CIntegerRanges-method.html</li>
	<li>coerce+2CGAlignments+2CIntegerRangesList-method.html</li>
	<li>coerce+2CGAlignmentsList+2CGAlignmentPairs-method.html</li>
	<li>coerce+2CGAlignmentsList+2CGRanges-method.html</li>
	<li>coerce+2CGAlignmentsList+2CGRangesList-method.html</li>
	<li>coerce+2CGAlignmentsList+2CIntegerRanges-method.html</li>
	<li>coerce+2CGAlignmentsList+2CIntegerRangesList-method.html</li>
	<li>coerce+2CGenomicRanges+2CGAlignments-method.html</li>
	<li>coerce+2Clist+2CGAlignmentsList-method.html</li>
	<li>coordinate-mapping-methods.html</li>
	<li>countCompatibleOverlaps.html</li>
	<li>countDumpedAlignments.html</li>
	<li>coverage+2CBamFile-method.html</li>
	<li>coverage+2CGAlignmentPairs-method.html</li>
	<li>coverage+2CGAlignments-method.html</li>
	<li>coverage+2CGAlignmentsList-method.html</li>
	<li>coverage+2Ccharacter-method.html</li>
	<li>coverage-methods.html</li>
	<li>coverage.html</li>
	<li>elementMetadata+2CGAlignmentsList-method.html</li>
	<li>elementMetadata+3C-+2CGAlignmentsList-method.html</li>
	<li>encodeOverlaps+2CGRangesList+2CGRangesList-method.html</li>
	<li>encodeOverlaps+2CIntegerRanges+2CIntegerRangesList-method.html</li>
	<li>encodeOverlaps+2CIntegerRangesList+2CIntegerRanges-method.html</li>
	<li>encodeOverlaps+2CIntegerRangesList+2CIntegerRangesList-method.html</li>
	<li>encodeOverlaps-methods.html</li>
	<li>encodeOverlaps.html</li>
	<li>encodeOverlaps1.html</li>
	<li>encoding+2COverlapEncodings-method.html</li>
	<li>encodingHalves+2COverlapEncodings-method.html</li>
	<li>encodingHalves+2Ccharacter-method.html</li>
	<li>encodingHalves+2Cfactor-method.html</li>
	<li>encodingHalves.html</li>
	<li>explodeCigarOpLengths.html</li>
	<li>explodeCigarOps.html</li>
	<li>extractAlignmentRangesOnReference.html</li>
	<li>extractQueryStartInTranscript.html</li>
	<li>extractSkippedExonRanks+2COverlapEncodings-method.html</li>
	<li>extractSkippedExonRanks+2Ccharacter-method.html</li>
	<li>extractSkippedExonRanks+2Cfactor-method.html</li>
	<li>extractSkippedExonRanks.html</li>
	<li>extractSpannedExonRanks+2COverlapEncodings-method.html</li>
	<li>extractSpannedExonRanks+2Ccharacter-method.html</li>
	<li>extractSpannedExonRanks+2Cfactor-method.html</li>
	<li>extractSpannedExonRanks.html</li>
	<li>extractSteppedExonRanks+2COverlapEncodings-method.html</li>
	<li>extractSteppedExonRanks+2Ccharacter-method.html</li>
	<li>extractSteppedExonRanks+2Cfactor-method.html</li>
	<li>extractSteppedExonRanks.html</li>
	<li>findCompatibleOverlaps+2CGAlignmentPairs+2CGRangesList-method.html</li>
	<li>findCompatibleOverlaps+2CGAlignments+2CGRangesList-method.html</li>
	<li>findCompatibleOverlaps-methods.html</li>
	<li>findCompatibleOverlaps.html</li>
	<li>findMateAlignment.html</li>
	<li>findOverlaps+2CGAlignmentPairs+2CGAlignmentPairs-method.html</li>
	<li>findOverlaps+2CGAlignmentPairs+2CVector-method.html</li>
	<li>findOverlaps+2CGAlignments+2CGAlignments-method.html</li>
	<li>findOverlaps+2CGAlignments+2CVector-method.html</li>
	<li>findOverlaps+2CGAlignmentsList+2CGAlignmentsList-method.html</li>
	<li>findOverlaps+2CGAlignmentsList+2CVector-method.html</li>
	<li>findOverlaps+2CVector+2CGAlignmentPairs-method.html</li>
	<li>findOverlaps+2CVector+2CGAlignments-method.html</li>
	<li>findOverlaps+2CVector+2CGAlignmentsList-method.html</li>
	<li>findOverlaps-methods.html</li>
	<li>findOverlaps.html</li>
	<li>findSpliceOverlaps+2CBamFile+2CANY-method.html</li>
	<li>findSpliceOverlaps+2CGAlignmentPairs+2CGRangesList-method.html</li>
	<li>findSpliceOverlaps+2CGAlignments+2CGRangesList-method.html</li>
	<li>findSpliceOverlaps+2CGRangesList+2CGRangesList-method.html</li>
	<li>findSpliceOverlaps+2Ccharacter+2CANY-method.html</li>
	<li>findSpliceOverlaps-methods.html</li>
	<li>findSpliceOverlaps.html</li>
	<li>first+2CGAlignmentPairs-method.html</li>
	<li>first.html</li>
	<li>flipQuery.html</li>
	<li>flippedQuery+2COverlapEncodings-method.html</li>
	<li>flippedQuery.html</li>
	<li>flushDumpedAlignments.html</li>
	<li>getDumpedAlignments.html</li>
	<li>getListElement+2CGAlignments-method.html</li>
	<li>granges+2CGAlignmentPairs-method.html</li>
	<li>granges+2CGAlignments-method.html</li>
	<li>granges+2CGAlignmentsList-method.html</li>
	<li>grglist+2CGAlignmentPairs-method.html</li>
	<li>grglist+2CGAlignments-method.html</li>
	<li>grglist+2CGAlignmentsList-method.html</li>
	<li>intra-range-methods.html</li>
	<li>invertStrand+2CGAlignmentPairs-method.html</li>
	<li>is.unsorted+2CGAlignments-method.html</li>
	<li>isCompatibleWithSkippedExons+2COverlapEncodings-method.html</li>
	<li>isCompatibleWithSkippedExons+2Ccharacter-method.html</li>
	<li>isCompatibleWithSkippedExons+2Cfactor-method.html</li>
	<li>isCompatibleWithSkippedExons.html</li>
	<li>isCompatibleWithSplicing+2COverlapEncodings-method.html</li>
	<li>isCompatibleWithSplicing+2Ccharacter-method.html</li>
	<li>isCompatibleWithSplicing+2Cfactor-method.html</li>
	<li>isCompatibleWithSplicing.html</li>
	<li>isProperPair+2CGAlignmentPairs-method.html</li>
	<li>isProperPair.html</li>
	<li>junctions+2CGAlignmentPairs-method.html</li>
	<li>junctions+2CGAlignments-method.html</li>
	<li>junctions+2CGAlignmentsList-method.html</li>
	<li>junctions-methods.html</li>
	<li>junctions.html</li>
	<li>last+2CGAlignmentPairs-method.html</li>
	<li>last.html</li>
	<li>levels+2COverlapEncodings-method.html</li>
	<li>levels.OverlapEncodings.html</li>
	<li>makeGAlignmentPairs.html</li>
	<li>mapFromAlignments+2CGenomicRanges+2CGAlignments-method.html</li>
	<li>mapFromAlignments+2CIntegerRanges+2CGAlignments-method.html</li>
	<li>mapFromAlignments.html</li>
	<li>mapToAlignments+2CGenomicRanges+2CGAlignments-method.html</li>
	<li>mapToAlignments+2CIntegerRanges+2CGAlignments-method.html</li>
	<li>mapToAlignments.html</li>
	<li>names+2CGAlignmentPairs-method.html</li>
	<li>names+2CGAlignments-method.html</li>
	<li>names+2CGAlignmentsList-method.html</li>
	<li>names+3C-+2CGAlignmentPairs-method.html</li>
	<li>names+3C-+2CGAlignments-method.html</li>
	<li>names+3C-+2CGAlignmentsList-method.html</li>
	<li>narrow+2CGappedReads-method.html</li>
	<li>njunc+2CANY-method.html</li>
	<li>njunc+2CGAlignmentPairs-method.html</li>
	<li>njunc+2CGAlignments-method.html</li>
	<li>njunc+2CGAlignmentsList-method.html</li>
	<li>njunc.html</li>
	<li>order+2CGAlignments-method.html</li>
	<li>parallel_slot_names+2CGAlignmentPairs-method.html</li>
	<li>parallel_slot_names+2CGAlignments-method.html</li>
	<li>parallel_slot_names+2COverlapEncodings-method.html</li>
	<li>paths.rds</li>
	<li>pcompare+2CGAlignments+2CGAlignments-method.html</li>
	<li>pileLettersAt.html</li>
	<li>pintersect+2CGAlignments+2CGRanges-method.html</li>
	<li>pintersect+2CGRanges+2CGAlignments-method.html</li>
	<li>pintersect.html</li>
	<li>pmapFromAlignments+2CGenomicRanges+2CGAlignments-method.html</li>
	<li>pmapFromAlignments+2CIntegerRanges+2CGAlignments-method.html</li>
	<li>pmapFromAlignments.html</li>
	<li>pmapToAlignments+2CGenomicRanges+2CGAlignments-method.html</li>
	<li>pmapToAlignments+2CIntegerRanges+2CGAlignments-method.html</li>
	<li>pmapToAlignments.html</li>
	<li>qnarrow+2CGAlignments-method.html</li>
	<li>qnarrow+2CGAlignmentsList-method.html</li>
	<li>qnarrow+2CGappedReads-method.html</li>
	<li>qnarrow.html</li>
	<li>qseq+2CGappedReads-method.html</li>
	<li>qseq.html</li>
	<li>queryLoc2refLoc.html</li>
	<li>queryLocs2refLocs.html</li>
	<li>qwidth+2CGAlignments-method.html</li>
	<li>qwidth+2CGAlignmentsList-method.html</li>
	<li>qwidth+2CGappedReads-method.html</li>
	<li>qwidth.html</li>
	<li>ranges+2CGAlignmentPairs-method.html</li>
	<li>ranges+2CGAlignments-method.html</li>
	<li>ranges+2CGAlignmentsList-method.html</li>
	<li>rank+2CGAlignments-method.html</li>
	<li>readGAlignmentPairs+2CBamFile-method.html</li>
	<li>readGAlignmentPairs+2Ccharacter-method.html</li>
	<li>readGAlignmentPairs.html</li>
	<li>readGAlignments+2CBamFile-method.html</li>
	<li>readGAlignments+2CBamViews-method.html</li>
	<li>readGAlignments+2Ccharacter-method.html</li>
	<li>readGAlignments.html</li>
	<li>readGAlignmentsList+2CBamFile-method.html</li>
	<li>readGAlignmentsList+2Ccharacter-method.html</li>
	<li>readGAlignmentsList.html</li>
	<li>readGappedReads+2CBamFile-method.html</li>
	<li>readGappedReads+2Ccharacter-method.html</li>
	<li>readGappedReads.html</li>
	<li>readSTARJunctions.html</li>
	<li>readTopHatJunctions.html</li>
	<li>relistToClass+2CGAlignments-method.html</li>
	<li>rglist+2CGAlignments-method.html</li>
	<li>rglist+2CGAlignmentsList-method.html</li>
	<li>rname+2CGAlignments-method.html</li>
	<li>rname+2CGAlignmentsList-method.html</li>
	<li>rname+3C-+2CGAlignments-method.html</li>
	<li>rname+3C-+2CGAlignmentsList-method.html</li>
	<li>rname+3C-.html</li>
	<li>rname.html</li>
	<li>second+2CGAlignmentPairs-method.html</li>
	<li>second.html</li>
	<li>selectEncodingWithCompatibleStrand.html</li>
	<li>seqinfo+2CGAlignmentPairs-method.html</li>
	<li>seqinfo+2CGAlignments-method.html</li>
	<li>seqinfo+2CGAlignmentsList-method.html</li>
	<li>seqinfo+3C-+2CGAlignmentPairs-method.html</li>
	<li>seqinfo+3C-+2CGAlignments-method.html</li>
	<li>seqinfo+3C-+2CGAlignmentsList-method.html</li>
	<li>seqlevelsInUse+2CGAlignmentPairs-method.html</li>
	<li>seqnames+2CGAlignmentPairs-method.html</li>
	<li>seqnames+2CGAlignments-method.html</li>
	<li>seqnames+2CGAlignmentsList-method.html</li>
	<li>seqnames+3C-+2CGAlignments-method.html</li>
	<li>seqnames+3C-+2CGAlignmentsList-method.html</li>
	<li>sequenceLayer.html</li>
	<li>setops-methods.html</li>
	<li>show+2CGAlignmentPairs-method.html</li>
	<li>show+2CGAlignments-method.html</li>
	<li>show+2CGAlignmentsList-method.html</li>
	<li>show+2COverlapEncodings-method.html</li>
	<li>sort+2CGAlignments-method.html</li>
	<li>stackStringsFromBam.html</li>
	<li>stackStringsFromGAlignments.html</li>
	<li>start+2CGAlignments-method.html</li>
	<li>strand+2CGAlignmentPairs-method.html</li>
	<li>strand+2CGAlignments-method.html</li>
	<li>strand+2CGAlignmentsList-method.html</li>
	<li>strand+3C-+2CGAlignments+2CANY-method.html</li>
	<li>strand+3C-+2CGAlignmentsList+2Ccharacter-method.html</li>
	<li>strand+3C-+2CGAlignmentsList-method.html</li>
	<li>strandMode+2CGAlignmentPairs-method.html</li>
	<li>strandMode+3C-+2CGAlignmentPairs-method.html</li>
	<li>strandMode+3C-.html</li>
	<li>strandMode.html</li>
	<li>summarizeJunctions.html</li>
	<li>summarizeOverlaps+2CBamViews+2Cmissing-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CBamFile-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CBamFileList-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CGAlignmentPairs-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CGAlignments-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CGAlignmentsList-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CGRanges-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2CGRangesList-method.html</li>
	<li>summarizeOverlaps+2CGRanges+2Ccharacter-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CBamFile-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CBamFileList-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CGAlignmentPairs-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CGAlignments-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CGAlignmentsList-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CGRanges-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2CGRangesList-method.html</li>
	<li>summarizeOverlaps+2CGRangesList+2Ccharacter-method.html</li>
	<li>summarizeOverlaps-methods.html</li>
	<li>summarizeOverlaps.html</li>
	<li>unlist+2CGAlignmentPairs-method.html</li>
	<li>update+2CGAlignments-method.html</li>
	<li>updateObject+2CGAlignments-method.html</li>
	<li>validCigar.html</li>
	<li>width+2CGAlignments-method.html</li>
	<li>windows+2CGAlignments-method.html</li>
	</ul></li>
	<li>html/<ul>
	<li>00Index.html</li>
	<li>GAlignmentPairs-class.html</li>
	<li>GAlignments-class.html</li>
	<li>GAlignmentsList-class.html</li>
	<li>GappedReads-class.html</li>
	<li>OverlapEncodings-class.html</li>
	<li>R.css</li>
	<li>cigar-utils.html</li>
	<li>coordinate-mapping-methods.html</li>
	<li>coverage-methods.html</li>
	<li>encodeOverlaps-methods.html</li>
	<li>findCompatibleOverlaps-methods.html</li>
	<li>findMateAlignment.html</li>
	<li>findOverlaps-methods.html</li>
	<li>findSpliceOverlaps-methods.html</li>
	<li>intra-range-methods.html</li>
	<li>junctions-methods.html</li>
	<li>pileLettersAt.html</li>
	<li>readGAlignments.html</li>
	<li>sequenceLayer.html</li>
	<li>setops-methods.html</li>
	<li>stackStringsFromGAlignments.html</li>
	<li>summarizeOverlaps-methods.html</li>
	</ul></li>
	<li>libs/<ul>
	<li>GenomicAlignments.so</li>
	</ul></li>
	<li>unitTests/<ul>
	<li>test_GAlignments-class.R</li>
	<li>test_GAlignmentsList-class.R</li>
	<li>test_cigar-utils.R</li>
	<li>test_coordinate-mapping-methods.R</li>
	<li>test_findSpliceOverlaps-methods.R</li>
	<li>test_intra-range-methods.R</li>
	<li>test_readGAlignmentPairs.R</li>
	<li>test_readGAlignments.R</li>
	<li>test_readGAlignmentsList.R</li>
	<li>test_summarizeOverlaps-methods.R</li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
</div>

  <div class="text-muted text-center">
    <br>
    Sources on <a href="https://pagure.io/fedora-packages-static">Pagure</a>
  </div>
</div>