<!DOCTYPE html>
<html>
	<head>
		<title>R-Biostrings - Details</title>
		<link rel="stylesheet" href="../../assets/css/bootstrap.min.css">
		<link rel="stylesheet" href="../../assets/css/main.css">
		<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
	</head>
	<body>
		<div class="container">
			<div>
				<div class="row">
					<a href="/" class="col-md-6">
						<div>
							<img src="../../../assets/images/logo.png" style="margin-top: -30px;">
						</div>
					</a>
					<div class="col-md-6">
						
					</div>
				</div>
			</div>

			<hr />

			<h1>
				R-Biostrings
				<small class="text-muted"> in Fedora 33</small>
			</h1>
			<p>
			This page displays detailed information for a specific version of R-Biostrings. You might
			want go back to <a href=".">the main page of R-Biostrings</a>.
			</p>

			<h2>Changelog</h2>
			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Date</th>
						<th scope="col">Author</th>
						<th scope="col">Change</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>2020-07-27</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 2.56.0-2</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2020-06-08</td>
					<td>Tom Callaway <spot at fedoraproject dot org> - 2.56.0-1</td>
					<td>- update to 2.56.0
- rebuild for R 4</td>
				</tr>
				
				<tr scope="row">
					<td>2020-01-28</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 2.54.0-2</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-11-04</td>
					<td>Tom Callaway <spot at fedoraproject dot org> - 2.54.0-1</td>
					<td>- update to 2.54.0</td>
				</tr>
				
				<tr scope="row">
					<td>2019-08-12</td>
					<td>Elliott Sales de Andrade <quantum dot analyst at gmail dot com> - 2.48.0-5</td>
					<td>- Remove explicit dependencies provided by automatic dependency generator</td>
				</tr>
				
				<tr scope="row">
					<td>2019-07-24</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 2.48.0-4</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_31_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-01-31</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 2.48.0-3</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_30_Mass_Rebuild</td>
				</tr>
				
				<tbody>
				</tbody>
			</table>

			<h2>Files</h2>

			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Type</th>
						<th scope="col">Path</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/DESCRIPTION</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/INDEX</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/Meta</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/NAMESPACE</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/NEWS</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/R</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/data</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/doc</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/extdata</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/help</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/html</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/libs</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/help/AnIndex</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/help/Biostrings.rdb</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/help/Biostrings.rdx</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/help/aliases.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/help/paths.rds</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/.build-id/73</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_MultipleAlignment.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_XString.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_XStringSet.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_XStringSetList.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_findPalindromes.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_matchDNAPattern.R.FIXME</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_matchPDict.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_pairwiseAlignment.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/unitTests/test_twoWayAlphabetFrequency.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/Rd.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/data.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/features.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/hsearch.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/links.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/nsInfo.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/package.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/Meta/vignette.rds</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/R/Biostrings</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/R/Biostrings.rdb</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/R/Biostrings.rdx</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/Biostrings2Classes.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/Biostrings2Classes.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/Biostrings2Classes.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/BiostringsQuickOverview.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/BiostringsQuickOverview.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/MultipleAlignments.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/MultipleAlignments.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/MultipleAlignments.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/PairwiseAlignments.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/PairwiseAlignments.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/PairwiseAlignments.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/index.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/matchprobes.R</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/matchprobes.Rnw</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/doc/matchprobes.pdf</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib64/R/library/Biostrings</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/BLOSUM100.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/BLOSUM45.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/BLOSUM50.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/BLOSUM62.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/BLOSUM80.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/HNF4alpha.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/PAM120.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/PAM250.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/PAM30.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/PAM40.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/PAM70.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/phiX174Phage.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/srPhiX174.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/data/yeastSEQCHR1.rda</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/.build-id/73/83e8c83c60c2f4123d7f8919e8788e3c76432e</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/.build-id</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/libs/Biostrings.so</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/Phylip.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/README</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/Sc.fa</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/Sp.fa</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/dm3_upstream2000.fa.gz</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/fastaEx.fa</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/hg18chr8_gap.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/hg18chrY_gap.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/msx2_mRNA.aln</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/s_1_sequence.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/someORF.fa</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/extdata/someORF.fa.gz</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/00Index.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/AAString-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/AMINO_ACID_CODE.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/AlignedXStringSet-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/Biostrings-internals.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/DNAString-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/GENETIC_CODE.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/HNF4alpha.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/IUPAC_CODE_MAP.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/InDel-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/MIndex-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/MaskedXString-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/MultipleAlignment-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/PDict-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/PairwiseAlignments-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/PairwiseAlignments-io.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/QualityScaledXStringSet-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/R.css</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/RNAString-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XString-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringPartialMatches-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringQuality-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringSet-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringSet-comparison.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringSet-io.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringSetList-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/XStringViews-class.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/align-utils.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/chartr.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/detail.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/dinucleotideFrequencyTest.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/findPalindromes.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/getSeq.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/gregexpr2.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/injectHardMask.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/letter.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/letterFrequency.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/longestConsecutive.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/lowlevel-matching.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/maskMotif.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/match-utils.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchLRPatterns.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchPDict-exact.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchPDict-inexact.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchPWM.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchPattern.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchProbePair.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/matchprobes.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/misc.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/needwunsQS.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/nucleotideFrequency.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/padAndClip.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/pairwiseAlignment.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/phiX174Phage.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/pid.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/pmatchPattern.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/replaceAt.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/replaceLetterAt.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/reverseComplement.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/stringDist.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/substitution_matrices.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/toComplex.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/translate.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/trimLRPatterns.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/xscat.html</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib64/R/library/Biostrings/html/yeastSEQCHR1.html</td>
				</tr>
				
				<tbody>
		</div>
	</body>
</html>