<!DOCTYPE html>

<title>R-Rsamtools-2.12.0-1.fc37 - Fedora Packages</title>
<link rel="stylesheet" href="../../../assets/css/bootstrap.min.css">
<link rel="stylesheet" href="../../../assets/css/main.css">
<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta charset="UTF-8">
<meta name="description" content="View R-Rsamtools-2.12.0-1.fc37 in Fedora 37. R-Rsamtools: R interface to samtools">

<div class="container">
  <div class="row">
    <a href="../../../" class="col-md-6 d-flex align-items-center justify-content-center justify-content-md-start p-1">
      <img alt="Fedora Packages" src="../../../assets/images/logo.png">
    </a>
    <div class="col-md-6 d-flex align-items-center">
    </div>
  </div>

  <hr />

<h1>
	R-Rsamtools-2.12.0-1.fc37<small class="text-muted"> in Fedora 37</small>
</h1>
<p>
	<a href=".">&crarr; Return to the main page of R-Rsamtools</a><br>
	<a href="https://koji.fedoraproject.org/koji/search?match=exact&type=build&terms=R-Rsamtools-2.12.0-1.fc37">View build</a><br>
	<a href="https://bodhi.fedoraproject.org/updates/?search=R-Rsamtools-2.12.0-1.fc37">Search for updates</a>
</p>

<p>
	<b>Package Info <span class="text-muted">(Data from x86_64 build)</span></b><br>
	<a href="#changelog">&#129047; Changelog</a><br>
	<a href="#dependencies">&#129047; Dependencies</a><br>
	<a href="#provides">&#129047; Provides</a><br>
	<a href="#files">&#129047; Files</a><br>
</p>

<h2 id="changelog">Changelog</h2>
<div class="table-responsive">
	<table class="table table-striped table-borderless ">
		<thead>
			<tr>
				<th scope="col">Date</th>
				<th scope="col">Author</th>
				<th scope="col">Change</th>
			</tr>
		</thead>
		<tr>
			<td>2022-09-03</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.12.0-1</td>
			<td>- update to 2.12.0
- rebuild for R 4.2.1</td>
		</tr>
		<tr>
			<td>2022-07-20</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.8.0-4</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_37_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2022-01-19</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.8.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_36_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-07-21</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.8.0-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_35_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-06-16</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.8.0-1</td>
			<td>- update to 2.8.0
- Rebuilt for R 4.1.0</td>
		</tr>
		<tr>
			<td>2021-02-04</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.6.0-1</td>
			<td>- update to 2.6.0</td>
		</tr>
		<tr>
			<td>2021-01-25</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.4.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
		</tr>
		<tbody>
		</tbody>
	</table>
</div>

<div class="row">
	<div class="col">
		<h2 id="dependencies">Dependencies</h2>
		<ul>
			<li>
			<a href="/pkgs/R-BiocGenerics/R-BiocGenerics/fedora-37.html">
			R-BiocGenerics</a> &gt;= 0.25.1

			</li>
			<li>
			<a href="/pkgs/R-BiocParallel/R-BiocParallel/fedora-37.html">
			R-BiocParallel</a>

			</li>
			<li>
			<a href="/pkgs/R-Biostrings/R-Biostrings/fedora-37.html">
			R-Biostrings</a> &gt;= 2.47.6

			</li>
			<li>
			<a href="/pkgs/R-GenomeInfoDb/R-GenomeInfoDb/fedora-37.html">
			R-GenomeInfoDb</a> &gt;= 1.1.3

			</li>
			<li>
			<a href="/pkgs/R-GenomicRanges/R-GenomicRanges/fedora-37.html">
			R-GenomicRanges</a> &gt;= 1.31.8

			</li>
			<li>
			<a href="/pkgs/R-IRanges/R-IRanges/fedora-37.html">
			R-IRanges</a> &gt;= 2.13.12

			</li>
			<li>
			<a href="/pkgs/R-S4Vectors/R-S4Vectors/fedora-37.html">
			R-S4Vectors</a> &gt;= 0.17.25

			</li>
			<li>
			<a href="/pkgs/R-XVector/R-XVector/fedora-37.html">
			R-XVector</a> &gt;= 0.19.7

			</li>
			<li>
			<a href="/pkgs/R-bitops/R-bitops/fedora-37.html">
			R-bitops</a>

			</li>
			<li>
			<a href="/pkgs/R/R-core/fedora-37.html">
			R-core</a> = 4.2

			</li>
			<li>
			<a href="/pkgs/glibc/glibc/fedora-37.html">
			glibc</a>

			</li>
			<li>
			<a href="/pkgs/curl/libcurl/fedora-37.html">
			libcurl</a>

			</li>
			<li>
			<a href="/pkgs/curl/libcurl-minimal/fedora-37.html">
			libcurl-minimal</a>

			</li>
			<li>
			<a href="/pkgs/gcc/libgcc/fedora-37.html">
			libgcc</a>

			</li>
			<li>
			<a href="/pkgs/gcc/libstdc++/fedora-37.html">
			libstdc++</a>

			</li>
		</ul>
	</div>
	<div class="col">
		<h2 id="provides">Provides</h2>
		<ul>
			<li>R(Rsamtools)</li>
			<li>R-Rsamtools</li>
			<li>R-Rsamtools(x86-64)</li>
			<li>R-Rsamtools-devel</li>
		</ul>
	</div>
</div>

<h2 id="files">Files</h2>

<div class="tree">
	<ul>
	<li>usr/<ul>
	<li>lib/<ul>
	<li>.build-id/<ul>
	<li>68/<ul>
	<li>e884365ceae4ab6612e7769cb335290c7f000f</li>
	</ul></li>
	</ul></li>
	</ul></li>
	<li>lib64/<ul>
	<li>R/<ul>
	<li>library/<ul>
	<li>Rsamtools/<ul>
	<li>DESCRIPTION</li>
	<li>INDEX</li>
	<li>LICENSE</li>
	<li>Meta/<ul>
	<li>Rd.rds</li>
	<li>features.rds</li>
	<li>hsearch.rds</li>
	<li>links.rds</li>
	<li>nsInfo.rds</li>
	<li>package.rds</li>
	<li>vignette.rds</li>
	</ul></li>
	<li>NAMESPACE</li>
	<li>NEWS</li>
	<li>R/<ul>
	<li>Rsamtools</li>
	<li>Rsamtools.rdb</li>
	<li>Rsamtools.rdx</li>
	</ul></li>
	<li>doc/<ul>
	<li>Rsamtools-Overview.R</li>
	<li>Rsamtools-Overview.Rnw</li>
	<li>Rsamtools-Overview.pdf</li>
	<li>index.html</li>
	</ul></li>
	<li>extdata/<ul>
	<li>ce2dict1.fa</li>
	<li>ce2dict1.fa.fai</li>
	<li>ex1.bam</li>
	<li>ex1.bam.bai</li>
	<li>ex1.bcf.gz</li>
	<li>ex1.bcf.gz.csi</li>
	<li>ex1.sam</li>
	<li>ex1.vcf.gz</li>
	<li>ex1.vcf.gz.csi</li>
	<li>example.gtf.gz</li>
	<li>example.gtf.gz.tbi</li>
	<li>example_from_SAM_Spec.bam</li>
	<li>example_from_SAM_Spec.bam.bai</li>
	<li>example_from_SAM_Spec.sam</li>
	<li>no_which_buffered_pileup.bam</li>
	<li>no_which_buffered_pileup.bam.bai</li>
	<li>no_which_buffered_pileup.sam</li>
	<li>no_which_whole_file.bam</li>
	<li>no_which_whole_file.bam.bai</li>
	<li>no_which_whole_file.sam</li>
	<li>olaps.Rda</li>
	<li>pileup.txt</li>
	<li>querybins.bam</li>
	<li>querybins.bam.bai</li>
	<li>querybins.sam</li>
	<li>revbins.bam</li>
	<li>revbins.bam.bai</li>
	<li>revbins.sam</li>
	<li>samtools-github.txt</li>
	<li>slxMaq09_urls.txt</li>
	<li>tagfilter.bam</li>
	<li>tagfilter.bam.bai</li>
	<li>tagfilter.sam</li>
	<li>tiny.bam</li>
	<li>tiny.bam.bai</li>
	<li>tiny.sam</li>
	<li>tophat/<ul>
	<li>README</li>
	<li>accepted_hits.sam</li>
	<li>coverage.wig</li>
	<li>junctions.bed</li>
	</ul></li>
	</ul></li>
	<li>help/<ul>
	<li>+5B+2CBamViews+2CANY+2CANY-method.html</li>
	<li>+5B+2CBamViews+2CANY+2Cmissing-method.html</li>
	<li>+5B+2CBamViews+2Cmissing+2CANY-method.html</li>
	<li>AnIndex</li>
	<li>ApplyPileupsParam-class.html</li>
	<li>ApplyPileupsParam.html</li>
	<li>BamFile-class.html</li>
	<li>BamFile.html</li>
	<li>BamFileList-class.html</li>
	<li>BamFileList.html</li>
	<li>BamSampler-class.html</li>
	<li>BamSampler.html</li>
	<li>BamViews+2CGRanges-method.html</li>
	<li>BamViews+2Cmissing-method.html</li>
	<li>BamViews-class.html</li>
	<li>BamViews.html</li>
	<li>BcfFile-class.html</li>
	<li>BcfFile.html</li>
	<li>BcfFileList-class.html</li>
	<li>BcfFileList.html</li>
	<li>FLAG_BITNAMES.html</li>
	<li>FaFile-class.html</li>
	<li>FaFile.html</li>
	<li>FaFileList-class.html</li>
	<li>FaFileList.html</li>
	<li>PileupFiles+2Ccharacter-method.html</li>
	<li>PileupFiles+2Clist-method.html</li>
	<li>PileupFiles-class.html</li>
	<li>PileupFiles.html</li>
	<li>PileupParam-class.html</li>
	<li>PileupParam.html</li>
	<li>Rsamtools-package.html</li>
	<li>Rsamtools.html</li>
	<li>Rsamtools.rdb</li>
	<li>Rsamtools.rdx</li>
	<li>RsamtoolsFile-class.html</li>
	<li>RsamtoolsFile.html</li>
	<li>RsamtoolsFileList-class.html</li>
	<li>RsamtoolsFileList.html</li>
	<li>ScanBVcfParam-class.html</li>
	<li>ScanBamParam+2CANY-method.html</li>
	<li>ScanBamParam+2CGRanges-method.html</li>
	<li>ScanBamParam+2CIntegerRangesList-method.html</li>
	<li>ScanBamParam+2Cmissing-method.html</li>
	<li>ScanBamParam-class.html</li>
	<li>ScanBamParam.html</li>
	<li>ScanBcfParam+2CGRanges-method.html</li>
	<li>ScanBcfParam+2CGRangesList-method.html</li>
	<li>ScanBcfParam+2CIntegerRangesList-method.html</li>
	<li>ScanBcfParam+2Cmissing-method.html</li>
	<li>ScanBcfParam-class.html</li>
	<li>ScanBcfParam.html</li>
	<li>TabixFile-class.html</li>
	<li>TabixFile.html</li>
	<li>TabixFileList-class.html</li>
	<li>TabixFileList.html</li>
	<li>aliases.rds</li>
	<li>applyPileups+2CPileupFiles+2CApplyPileupsParam-method.html</li>
	<li>applyPileups+2CPileupFiles+2Cmissing-method.html</li>
	<li>applyPileups.html</li>
	<li>asBam+2Ccharacter-method.html</li>
	<li>asBam.html</li>
	<li>asBcf+2Ccharacter-method.html</li>
	<li>asBcf.html</li>
	<li>asMates+2CBamFile-method.html</li>
	<li>asMates+2CBamFileList-method.html</li>
	<li>asMates+3C-+2CBamFile-method.html</li>
	<li>asMates+3C-+2CBamFileList-method.html</li>
	<li>asMates+3C-.html</li>
	<li>asMates.html</li>
	<li>asSam+2Ccharacter-method.html</li>
	<li>asSam.html</li>
	<li>bamDirname+3C-.html</li>
	<li>bamExperiment.html</li>
	<li>bamFlag+3C-.html</li>
	<li>bamFlag.html</li>
	<li>bamFlagAND.html</li>
	<li>bamFlagAsBitMatrix.html</li>
	<li>bamFlagTest.html</li>
	<li>bamIndicies.html</li>
	<li>bamMapqFilter+3C-.html</li>
	<li>bamMapqFilter.html</li>
	<li>bamPaths.html</li>
	<li>bamRanges+3C-.html</li>
	<li>bamRanges.html</li>
	<li>bamReverseComplement+3C-.html</li>
	<li>bamReverseComplement.html</li>
	<li>bamSamples+3C-.html</li>
	<li>bamSamples.html</li>
	<li>bamSimpleCigar+3C-.html</li>
	<li>bamSimpleCigar.html</li>
	<li>bamTag+3C-.html</li>
	<li>bamTag.html</li>
	<li>bamTagFilter+3C-.html</li>
	<li>bamTagFilter.html</li>
	<li>bamWhat+3C-.html</li>
	<li>bamWhat.html</li>
	<li>bamWhich+3C-+2CScanBamParam+2CANY-method.html</li>
	<li>bamWhich+3C-+2CScanBamParam+2CGRanges-method.html</li>
	<li>bamWhich+3C-+2CScanBamParam+2CIntegerRangesList-method.html</li>
	<li>bamWhich+3C-.html</li>
	<li>bamWhich.html</li>
	<li>bcfFixed.html</li>
	<li>bcfGeno.html</li>
	<li>bcfInfo.html</li>
	<li>bcfMode.html</li>
	<li>bcfSamples.html</li>
	<li>bcfTrimEmpty.html</li>
	<li>bcfWhich.html</li>
	<li>bgzip.html</li>
	<li>bgzipTabix.html</li>
	<li>bzfile-class.html</li>
	<li>close.BamFile.html</li>
	<li>close.BcfFile.html</li>
	<li>close.FaFile.html</li>
	<li>close.RsamtoolsFileList.html</li>
	<li>close.TabixFile.html</li>
	<li>countBam+2CBamFile-method.html</li>
	<li>countBam+2CBamFileList-method.html</li>
	<li>countBam+2CBamViews-method.html</li>
	<li>countBam+2Ccharacter-method.html</li>
	<li>countBam.html</li>
	<li>countFa+2CFaFile-method.html</li>
	<li>countFa+2Ccharacter-method.html</li>
	<li>countFa.html</li>
	<li>countTabix.html</li>
	<li>cycle_bins.html</li>
	<li>defunct.html</li>
	<li>deprecated.html</li>
	<li>dim+2CBamViews-method.html</li>
	<li>dimnames+2CBamViews-method.html</li>
	<li>dimnames+3C-+2CBamViews+2CANY-method.html</li>
	<li>dimnames+3C-+2CBamViews-method.html</li>
	<li>distinguish_nucleotides.html</li>
	<li>distinguish_strands.html</li>
	<li>fifo-class.html</li>
	<li>filterBam+2CBamFile-method.html</li>
	<li>filterBam+2Ccharacter-method.html</li>
	<li>filterBam.html</li>
	<li>getSeq+2CFaFile-method.html</li>
	<li>getSeq+2CFaFileList-method.html</li>
	<li>gzfile-class.html</li>
	<li>gzindex+2CFaFile-method.html</li>
	<li>gzindex+2CFaFileList-method.html</li>
	<li>gzindex+3C-+2CFaFile-method.html</li>
	<li>gzindex+3C-+2CFaFileList-method.html</li>
	<li>gzindex+3C-.html</li>
	<li>gzindex.html</li>
	<li>hasUnmappedMate.html</li>
	<li>headerTabix+2CTabixFile-method.html</li>
	<li>headerTabix+2Ccharacter-method.html</li>
	<li>headerTabix.html</li>
	<li>idxstatsBam+2CBamFile-method.html</li>
	<li>idxstatsBam+2Ccharacter-method.html</li>
	<li>idxstatsBam.html</li>
	<li>ignore_query_Ns.html</li>
	<li>include_deletions.html</li>
	<li>include_insertions.html</li>
	<li>index+2CRsamtoolsFile-method.html</li>
	<li>index+2CRsamtoolsFileList-method.html</li>
	<li>index+3C-+2CRsamtoolsFile-method.html</li>
	<li>index+3C-+2CRsamtoolsFileList-method.html</li>
	<li>index+3C-.html</li>
	<li>index.html</li>
	<li>indexBam+2CBamFile-method.html</li>
	<li>indexBam+2Ccharacter-method.html</li>
	<li>indexBam.html</li>
	<li>indexBcf+2CBcfFile-method.html</li>
	<li>indexBcf+2Ccharacter-method.html</li>
	<li>indexBcf.html</li>
	<li>indexFa+2CFaFile-method.html</li>
	<li>indexFa+2Ccharacter-method.html</li>
	<li>indexFa.html</li>
	<li>indexTabix.html</li>
	<li>isDuplicate.html</li>
	<li>isFirstMateRead.html</li>
	<li>isIncomplete+2CBamFile-method.html</li>
	<li>isMateMinusStrand.html</li>
	<li>isMinusStrand.html</li>
	<li>isNotPassingQualityControls.html</li>
	<li>isNotPrimaryRead.html</li>
	<li>isOpen+2CBamFile-method.html</li>
	<li>isOpen+2CBcfFile-method.html</li>
	<li>isOpen+2CFaFile-method.html</li>
	<li>isOpen+2CPileupFiles-method.html</li>
	<li>isOpen+2CRsamtoolsFile-method.html</li>
	<li>isOpen+2CRsamtoolsFileList-method.html</li>
	<li>isOpen+2CTabixFile-method.html</li>
	<li>isPaired.html</li>
	<li>isProperPair.html</li>
	<li>isSecondMateRead.html</li>
	<li>isSecondaryAlignment.html</li>
	<li>isUnmappedQuery.html</li>
	<li>left_bins.html</li>
	<li>max_depth.html</li>
	<li>mergeBam+2CBamFileList-method.html</li>
	<li>mergeBam+2Ccharacter-method.html</li>
	<li>mergeBam.html</li>
	<li>min_base_quality.html</li>
	<li>min_mapq.html</li>
	<li>min_minor_allele_depth.html</li>
	<li>min_nucleotide_depth.html</li>
	<li>names+2CBamViews-method.html</li>
	<li>names+2CRsamtoolsFileList-method.html</li>
	<li>names+3C-+2CBamViews-method.html</li>
	<li>obeyQname+2CBamFile-method.html</li>
	<li>obeyQname+2CBamFileList-method.html</li>
	<li>obeyQname+3C-+2CBamFile-method.html</li>
	<li>obeyQname+3C-+2CBamFileList-method.html</li>
	<li>obeyQname+3C-.html</li>
	<li>obeyQname.html</li>
	<li>open.BamFile.html</li>
	<li>open.BcfFile.html</li>
	<li>open.FaFile.html</li>
	<li>open.RsamtoolsFileList.html</li>
	<li>open.TabixFile.html</li>
	<li>path+2CRsamtoolsFile-method.html</li>
	<li>path+2CRsamtoolsFileList-method.html</li>
	<li>path.html</li>
	<li>paths.rds</li>
	<li>phred2ASCIIOffset.html</li>
	<li>pileup+2CBamFile-method.html</li>
	<li>pileup+2Ccharacter-method.html</li>
	<li>pileup.html</li>
	<li>pipe-class.html</li>
	<li>plpFiles.html</li>
	<li>plpFlag+3C-.html</li>
	<li>plpFlag.html</li>
	<li>plpMaxDepth+3C-.html</li>
	<li>plpMaxDepth.html</li>
	<li>plpMinBaseQuality+3C-.html</li>
	<li>plpMinBaseQuality.html</li>
	<li>plpMinDepth+3C-.html</li>
	<li>plpMinDepth.html</li>
	<li>plpMinMapQuality+3C-.html</li>
	<li>plpMinMapQuality.html</li>
	<li>plpParam.html</li>
	<li>plpWhat+3C-.html</li>
	<li>plpWhat.html</li>
	<li>plpWhich+3C-.html</li>
	<li>plpWhich.html</li>
	<li>plpYieldAll+3C-.html</li>
	<li>plpYieldAll.html</li>
	<li>plpYieldBy+3C-.html</li>
	<li>plpYieldBy.html</li>
	<li>plpYieldSize+3C-.html</li>
	<li>plpYieldSize.html</li>
	<li>qnamePrefixEnd+2CBamFile-method.html</li>
	<li>qnamePrefixEnd+2CBamFileList-method.html</li>
	<li>qnamePrefixEnd+3C-+2CBamFile-method.html</li>
	<li>qnamePrefixEnd+3C-+2CBamFileList-method.html</li>
	<li>qnamePrefixEnd+3C-.html</li>
	<li>qnamePrefixEnd.html</li>
	<li>qnameSuffixStart+2CBamFile-method.html</li>
	<li>qnameSuffixStart+2CBamFileList-method.html</li>
	<li>qnameSuffixStart+3C-+2CBamFile-method.html</li>
	<li>qnameSuffixStart+3C-+2CBamFileList-method.html</li>
	<li>qnameSuffixStart+3C-.html</li>
	<li>qnameSuffixStart.html</li>
	<li>query_bins.html</li>
	<li>quickBamCounts.html</li>
	<li>quickBamFlagSummary+2CBamFile-method.html</li>
	<li>quickBamFlagSummary+2Ccharacter-method.html</li>
	<li>quickBamFlagSummary+2Clist-method.html</li>
	<li>quickBamFlagSummary.html</li>
	<li>quickCountBam.html</li>
	<li>razip.html</li>
	<li>readPileup+2Ccharacter-method.html</li>
	<li>readPileup+2Cconnection-method.html</li>
	<li>readPileup.html</li>
	<li>scanBam+2CBamFile-method.html</li>
	<li>scanBam+2CBamSampler-method.html</li>
	<li>scanBam+2CBamViews-method.html</li>
	<li>scanBam+2Ccharacter-method.html</li>
	<li>scanBam.html</li>
	<li>scanBamFlag.html</li>
	<li>scanBamHeader+2CBamFile-method.html</li>
	<li>scanBamHeader+2Ccharacter-method.html</li>
	<li>scanBamHeader.html</li>
	<li>scanBamWhat.html</li>
	<li>scanBcf+2CBcfFile-method.html</li>
	<li>scanBcf+2Ccharacter-method.html</li>
	<li>scanBcf.html</li>
	<li>scanBcfHeader+2CBcfFile-method.html</li>
	<li>scanBcfHeader+2Ccharacter-method.html</li>
	<li>scanBcfHeader.html</li>
	<li>scanFa+2CFaFile+2CGRanges-method.html</li>
	<li>scanFa+2CFaFile+2CIntegerRangesList-method.html</li>
	<li>scanFa+2CFaFile+2Cmissing-method.html</li>
	<li>scanFa+2Ccharacter+2CGRanges-method.html</li>
	<li>scanFa+2Ccharacter+2CIntegerRangesList-method.html</li>
	<li>scanFa+2Ccharacter+2Cmissing-method.html</li>
	<li>scanFa.html</li>
	<li>scanFaIndex+2CFaFile-method.html</li>
	<li>scanFaIndex+2CFaFileList-method.html</li>
	<li>scanFaIndex+2Ccharacter-method.html</li>
	<li>scanFaIndex.html</li>
	<li>scanTabix+2CTabixFile+2CGRanges-method.html</li>
	<li>scanTabix+2CTabixFile+2CIntegerRangesList-method.html</li>
	<li>scanTabix+2CTabixFile+2Cmissing-method.html</li>
	<li>scanTabix+2Ccharacter+2CANY-method.html</li>
	<li>scanTabix+2Ccharacter+2CGRanges-method.html</li>
	<li>scanTabix+2Ccharacter+2CIntegerRangesList-method.html</li>
	<li>scanTabix+2Ccharacter+2Cmissing-method.html</li>
	<li>scanTabix.html</li>
	<li>seqinfo+2CBamFile-method.html</li>
	<li>seqinfo+2CBamFileList-method.html</li>
	<li>seqinfo+2CFaFile-method.html</li>
	<li>seqnamesTabix+2CTabixFile-method.html</li>
	<li>seqnamesTabix+2Ccharacter-method.html</li>
	<li>seqnamesTabix.html</li>
	<li>show+2CApplyPileupsParam-method.html</li>
	<li>show+2CBamFile-method.html</li>
	<li>show+2CBamFileList-method.html</li>
	<li>show+2CBamSampler-method.html</li>
	<li>show+2CBamViews-method.html</li>
	<li>show+2CFaFile-method.html</li>
	<li>show+2CPileupFiles-method.html</li>
	<li>show+2CPileupParam-method.html</li>
	<li>show+2CRsamtoolsFile-method.html</li>
	<li>show+2CScanBVcfParam-method.html</li>
	<li>show+2CScanBamParam-method.html</li>
	<li>sortBam+2CBamFile-method.html</li>
	<li>sortBam+2Ccharacter-method.html</li>
	<li>sortBam.html</li>
	<li>testPairedEndBam+2CBamFile-method.html</li>
	<li>testPairedEndBam+2Ccharacter-method.html</li>
	<li>testPairedEndBam.html</li>
	<li>unz-class.html</li>
	<li>url-class.html</li>
	<li>yieldSize+2CRsamtoolsFile-method.html</li>
	<li>yieldSize+2CRsamtoolsFileList-method.html</li>
	<li>yieldSize+3C-+2CRsamtoolsFile-method.html</li>
	<li>yieldSize+3C-+2CRsamtoolsFileList-method.html</li>
	<li>yieldSize+3C-.html</li>
	<li>yieldSize.html</li>
	<li>yieldTabix+2CTabixFile-method.html</li>
	<li>yieldTabix.html</li>
	<li>zip.html</li>
	</ul></li>
	<li>html/<ul>
	<li>00Index.html</li>
	<li>ApplyPileupsParam-class.html</li>
	<li>BamFile-class.html</li>
	<li>BamViews-class.html</li>
	<li>BcfFile-class.html</li>
	<li>FaFile-class.html</li>
	<li>PileupFiles-class.html</li>
	<li>R.css</li>
	<li>Rsamtools-package.html</li>
	<li>RsamtoolsFile-class.html</li>
	<li>RsamtoolsFileList-class.html</li>
	<li>ScanBamParam-class.html</li>
	<li>ScanBcfParam-class.html</li>
	<li>TabixFile-class.html</li>
	<li>applyPileups.html</li>
	<li>defunct.html</li>
	<li>deprecated.html</li>
	<li>headerTabix.html</li>
	<li>indexTabix.html</li>
	<li>pileup.html</li>
	<li>quickBamFlagSummary.html</li>
	<li>readPileup.html</li>
	<li>scanBam.html</li>
	<li>scanBcf.html</li>
	<li>scanFa.html</li>
	<li>scanTabix.html</li>
	<li>seqnamesTabix.html</li>
	<li>testPairedEndBam.html</li>
	<li>zip.html</li>
	</ul></li>
	<li>libs/<ul>
	<li>Rsamtools.so</li>
	</ul></li>
	<li>scripts/<ul>
	<li>BamViews-1000g.R</li>
	<li>features.R</li>
	<li>remote_test.R</li>
	<li>update-samtools.sh</li>
	</ul></li>
	<li>unitTests/<ul>
	<li>cases/<ul>
	<li>RNEXT.bam</li>
	<li>ex1.sam.gz</li>
	<li>ex1_noindex.bam</li>
	<li>ex1_shuf1000.bam</li>
	<li>ex1_shuf1000.bam.bai</li>
	<li>ex1_unsort.bam</li>
	<li>ex1_zero_index.bam.bai</li>
	<li>no_SAMPLE_header.vcf.gz</li>
	<li>no_header_line.vcf.gz</li>
	<li>pileup-no-stars.txt</li>
	<li>plp_refskip.bam</li>
	<li>plp_refskip.bam.bai</li>
	</ul></li>
	<li>test_BamFile.R</li>
	<li>test_BamViews.R</li>
	<li>test_BcfFile.R</li>
	<li>test_FaFile.R</li>
	<li>test_RsamtoolsFile.R</li>
	<li>test_TabixFile.R</li>
	<li>test_applyPileups.R</li>
	<li>test_asBam.R</li>
	<li>test_bam.R</li>
	<li>test_bam_count.R</li>
	<li>test_bam_header.R</li>
	<li>test_compression.R</li>
	<li>test_mapqfilter.R</li>
	<li>test_phred2ASCIIOffset.R</li>
	<li>test_pileup_nowhich.R</li>
	<li>test_pileup_querybins.R</li>
	<li>test_pileup_revbins.R</li>
	<li>test_pileup_single_range.R</li>
	<li>test_readPileup.R</li>
	<li>test_scanBamFlag.R</li>
	<li>test_sortBam_test.R</li>
	<li>test_tagfilter.R</li>
	<li>test_testPairedEndBam.R</li>
	<li>test_utilities.R</li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
</div>

  <div class="text-muted text-center">
    <br>
    Sources on <a href="https://pagure.io/fedora-packages-static">Pagure</a>
  </div>
</div>