<!DOCTYPE html>

<title>R-BSgenome-1.60.0-3.fc36 - Fedora Packages</title>
<link rel="stylesheet" href="../../../assets/css/bootstrap.min.css">
<link rel="stylesheet" href="../../../assets/css/main.css">
<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta charset="UTF-8">
<meta name="description" content="View R-BSgenome-1.60.0-3.fc36 in Fedora 36. R-BSgenome: Infrastructure for Biostrings-based genome data packages">

<div class="container">
  <div class="row">
    <a href="../../../" class="col-md-6 d-flex align-items-center justify-content-center justify-content-md-start p-1">
      <img alt="Fedora Packages" src="../../../assets/images/logo.png">
    </a>
    <div class="col-md-6 d-flex align-items-center">
    </div>
  </div>

  <hr />

<h1>
	R-BSgenome-1.60.0-3.fc36<small class="text-muted"> in Fedora 36</small>
</h1>
<p>
	<a href=".">&crarr; Return to the main page of R-BSgenome</a><br>
	<a href="https://koji.fedoraproject.org/koji/search?match=exact&type=build&terms=R-BSgenome-1.60.0-3.fc36">View build</a><br>
	<a href="https://bodhi.fedoraproject.org/updates/?search=R-BSgenome-1.60.0-3.fc36">Search for updates</a>
</p>

<p>
	<b>Package Info</b><br>
	<a href="#changelog">&#129047; Changelog</a><br>
	<a href="#dependencies">&#129047; Dependencies</a><br>
	<a href="#provides">&#129047; Provides</a><br>
	<a href="#files">&#129047; Files</a><br>
</p>

<h2 id="changelog">Changelog</h2>
<div class="table-responsive">
	<table class="table table-striped table-borderless ">
		<thead>
			<tr>
				<th scope="col">Date</th>
				<th scope="col">Author</th>
				<th scope="col">Change</th>
			</tr>
		</thead>
		<tr>
			<td>2022-01-19</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.60.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_36_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-07-21</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.60.0-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_35_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-06-17</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.60.0-1</td>
			<td>- update to 1.60.0
- Rebuilt for R 4.1.0</td>
		</tr>
		<tr>
			<td>2021-02-03</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.58.0-1</td>
			<td>- update to 1.58.0</td>
		</tr>
		<tr>
			<td>2021-01-25</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.56.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2020-07-27</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.56.0-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2020-06-08</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 1.56.0-1</td>
			<td>- update to 1.56.0
- rebuild for R 4</td>
		</tr>
		<tr>
			<td>2020-01-28</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 1.54.0-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
		</tr>
		<tbody>
		</tbody>
	</table>
</div>

<div class="row">
	<div class="col">
		<h2 id="dependencies">Dependencies</h2>
		<ul>
			<li>
			<a href="/pkgs/R-BiocGenerics/R-BiocGenerics/fedora-36.html">
			R-BiocGenerics</a> &gt;= 0.13.8

			</li>
			<li>
			<a href="/pkgs/R-Biostrings/R-Biostrings/fedora-36.html">
			R-Biostrings</a> &gt;= 2.47.6

			</li>
			<li>
			<a href="/pkgs/R-GenomeInfoDb/R-GenomeInfoDb/fedora-36.html">
			R-GenomeInfoDb</a> &gt;= 1.25.6

			</li>
			<li>
			<a href="/pkgs/R-GenomicRanges/R-GenomicRanges/fedora-36.html">
			R-GenomicRanges</a> &gt;= 1.31.10

			</li>
			<li>
			<a href="/pkgs/R-IRanges/R-IRanges/fedora-36.html">
			R-IRanges</a> &gt;= 2.13.16

			</li>
			<li>
			<a href="/pkgs/R-Rsamtools/R-Rsamtools/fedora-36.html">
			R-Rsamtools</a>

			</li>
			<li>
			<a href="/pkgs/R-S4Vectors/R-S4Vectors/fedora-36.html">
			R-S4Vectors</a> &gt;= 0.17.28

			</li>
			<li>
			<a href="/pkgs/R-XVector/R-XVector/fedora-36.html">
			R-XVector</a> &gt;= 0.29.3

			</li>
			<li>
			<a href="/pkgs/R/R-core/fedora-36.html">
			R-core</a> = 4.1

			</li>
			<li>
			<a href="/pkgs/R-matrixStats/R-matrixStats/fedora-36.html">
			R-matrixStats</a>

			</li>
			<li>
			<a href="/pkgs/R-rtracklayer/R-rtracklayer/fedora-36.html">
			R-rtracklayer</a> &gt;= 1.39.7

			</li>
		</ul>
	</div>
	<div class="col">
		<h2 id="provides">Provides</h2>
		<ul>
			<li>R(BSgenome)</li>
			<li>R-BSgenome</li>
		</ul>
	</div>
</div>

<h2 id="files">Files</h2>

<div class="tree">
	<ul>
	<li>usr/<ul>
	<li>share/<ul>
	<li>R/<ul>
	<li>library/<ul>
	<li>BSgenome/<ul>
	<li>DESCRIPTION</li>
	<li>INDEX</li>
	<li>Meta/<ul>
	<li>Rd.rds</li>
	<li>features.rds</li>
	<li>hsearch.rds</li>
	<li>links.rds</li>
	<li>nsInfo.rds</li>
	<li>package.rds</li>
	<li>vignette.rds</li>
	</ul></li>
	<li>NAMESPACE</li>
	<li>NEWS</li>
	<li>R/<ul>
	<li>BSgenome</li>
	<li>BSgenome.rdb</li>
	<li>BSgenome.rdx</li>
	</ul></li>
	<li>doc/<ul>
	<li>BSgenomeForge.R</li>
	<li>BSgenomeForge.Rnw</li>
	<li>BSgenomeForge.pdf</li>
	<li>GenomeSearching.R</li>
	<li>GenomeSearching.Rnw</li>
	<li>GenomeSearching.pdf</li>
	<li>index.html</li>
	</ul></li>
	<li>extdata/<ul>
	<li>GentlemanLab/<ul>
	<li>BSgenome.Amellifera.BeeBase.assembly4-seed</li>
	<li>BSgenome.Amellifera.BeeBase.assembly4-tools/<ul>
	<li>NOTES.TXT</li>
	<li>splitbigfasta.sh</li>
	</ul></li>
	<li>BSgenome.Amellifera.NCBI.AmelHAv3.1-seed</li>
	<li>BSgenome.Amellifera.NCBI.AmelHAv3.1-tools/<ul>
	<li>fasta_to_sorted_2bit.R</li>
	</ul></li>
	<li>BSgenome.Amellifera.UCSC.apiMel2-seed</li>
	<li>BSgenome.Amellifera.UCSC.apiMel2.masked-seed</li>
	<li>BSgenome.Athaliana.TAIR.04232008-seed</li>
	<li>BSgenome.Athaliana.TAIR.TAIR10.1-seed</li>
	<li>BSgenome.Athaliana.TAIR.TAIR10.1-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Athaliana.TAIR.TAIR9-seed</li>
	<li>BSgenome.Athaliana.TAIR.TAIR9-tools/<ul>
	<li>NOTES.TXT</li>
	</ul></li>
	<li>BSgenome.Btaurus.UCSC.bosTau3-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau3.masked-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau4-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau4.masked-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau6-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau6-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Btaurus.UCSC.bosTau6.masked-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau8-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau9-seed</li>
	<li>BSgenome.Btaurus.UCSC.bosTau9-tools/<ul>
	<li>sort_bosTau9.2bit.R</li>
	</ul></li>
	<li>BSgenome.Celegans.UCSC.ce10-seed</li>
	<li>BSgenome.Celegans.UCSC.ce11-seed</li>
	<li>BSgenome.Celegans.UCSC.ce2-seed</li>
	<li>BSgenome.Celegans.UCSC.ce6-seed</li>
	<li>BSgenome.Cfamiliaris.UCSC.canFam2-seed</li>
	<li>BSgenome.Cfamiliaris.UCSC.canFam2.masked-seed</li>
	<li>BSgenome.Cfamiliaris.UCSC.canFam3-seed</li>
	<li>BSgenome.Cfamiliaris.UCSC.canFam3-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Cfamiliaris.UCSC.canFam3.masked-seed</li>
	<li>BSgenome.Cjacchus.UCSC.calJac3-seed</li>
	<li>BSgenome.Cjacchus.UCSC.calJac3-tools/<ul>
	<li>sort_calJac3.2bit.R</li>
	</ul></li>
	<li>BSgenome.Creinhardtii.JGI.v5.6-seed</li>
	<li>BSgenome.Creinhardtii.JGI.v5.6-tools/<ul>
	<li>CITATION</li>
	<li>fasta_to_2bit.R</li>
	</ul></li>
	<li>BSgenome.Dmelanogaster.UCSC.dm2-seed</li>
	<li>BSgenome.Dmelanogaster.UCSC.dm2.masked-seed</li>
	<li>BSgenome.Dmelanogaster.UCSC.dm3-seed</li>
	<li>BSgenome.Dmelanogaster.UCSC.dm3.masked-seed</li>
	<li>BSgenome.Dmelanogaster.UCSC.dm6-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer10-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer10-tools/<ul>
	<li>sort_danRer10.2bit.R</li>
	</ul></li>
	<li>BSgenome.Drerio.UCSC.danRer11-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer11-tools/<ul>
	<li>sort_danRer11.2bit.R</li>
	</ul></li>
	<li>BSgenome.Drerio.UCSC.danRer5-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer5-tools/<ul>
	<li>NOTES.TXT</li>
	<li>splitbigfasta.sh</li>
	</ul></li>
	<li>BSgenome.Drerio.UCSC.danRer5.masked-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer6-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer6-tools/<ul>
	<li>NOTES.TXT</li>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Drerio.UCSC.danRer6.masked-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer7-seed</li>
	<li>BSgenome.Drerio.UCSC.danRer7-tools/<ul>
	<li>NOTES.TXT</li>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Drerio.UCSC.danRer7.masked-seed</li>
	<li>BSgenome.Dvirilis.Ensembl.dvircaf1-seed</li>
	<li>BSgenome.Ecoli.NCBI.20080805-seed</li>
	<li>BSgenome.Gaculeatus.UCSC.gasAcu1-seed</li>
	<li>BSgenome.Gaculeatus.UCSC.gasAcu1.masked-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal3-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal3.masked-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal4-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal4-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Ggallus.UCSC.galGal4.masked-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal5-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal5-tools/<ul>
	<li>sort_galGal5.2bit.R</li>
	</ul></li>
	<li>BSgenome.Ggallus.UCSC.galGal6-seed</li>
	<li>BSgenome.Ggallus.UCSC.galGal6-tools/<ul>
	<li>sort_galGal6.2bit.R</li>
	</ul></li>
	<li>BSgenome.Gmellonella.NCBI.ASM364042v2-seed</li>
	<li>BSgenome.Gmellonella.NCBI.ASM364042v2-tools/<ul>
	<li>fasta_to_sorted_2bit.R</li>
	</ul></li>
	<li>BSgenome.Hsapiens.NCBI.GRCh38-seed</li>
	<li>BSgenome.Hsapiens.NCBI.GRCh38-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Hsapiens.UCSC.hg17-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg17.masked-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg18-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg18.masked-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg19-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg19.masked-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg38-seed</li>
	<li>BSgenome.Hsapiens.UCSC.hg38.masked-seed</li>
	<li>BSgenome.Mdomestica.UCSC.monDom5-seed</li>
	<li>BSgenome.Mdomestica.UCSC.monDom5-tools/<ul>
	<li>sort_monDom5.2bit.R</li>
	</ul></li>
	<li>BSgenome.Mfascicularis.NCBI.5.0-seed</li>
	<li>BSgenome.Mfascicularis.NCBI.5.0-tools/<ul>
	<li>fasta_to_sorted2bit.R</li>
	</ul></li>
	<li>BSgenome.Mfuro.UCSC.musFur1-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac10-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac10-tools/<ul>
	<li>sort_rheMac10.2bit.R</li>
	</ul></li>
	<li>BSgenome.Mmulatta.UCSC.rheMac2-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac2.masked-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac3-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac3-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Mmulatta.UCSC.rheMac3.masked-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac8-seed</li>
	<li>BSgenome.Mmulatta.UCSC.rheMac8-tools/<ul>
	<li>sort_rheMac8.2bit.R</li>
	</ul></li>
	<li>BSgenome.Mmusculus.UCSC.mm10-seed</li>
	<li>BSgenome.Mmusculus.UCSC.mm10.masked-seed</li>
	<li>BSgenome.Mmusculus.UCSC.mm8-seed</li>
	<li>BSgenome.Mmusculus.UCSC.mm8.masked-seed</li>
	<li>BSgenome.Mmusculus.UCSC.mm9-seed</li>
	<li>BSgenome.Mmusculus.UCSC.mm9.masked-seed</li>
	<li>BSgenome.Ppaniscus.UCSC.panPan1-seed</li>
	<li>BSgenome.Ppaniscus.UCSC.panPan2-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro2-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro2.masked-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro3-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro3-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro3.masked-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro5-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro5-tools/<ul>
	<li>sort_panTro5.2bit.R</li>
	</ul></li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro6-seed</li>
	<li>BSgenome.Ptroglodytes.UCSC.panTro6-tools/<ul>
	<li>sort_panTro6.2bit.R</li>
	</ul></li>
	<li>BSgenome.Rnorvegicus.UCSC.rn4-seed</li>
	<li>BSgenome.Rnorvegicus.UCSC.rn4.masked-seed</li>
	<li>BSgenome.Rnorvegicus.UCSC.rn5-seed</li>
	<li>BSgenome.Rnorvegicus.UCSC.rn5-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Rnorvegicus.UCSC.rn5.masked-seed</li>
	<li>BSgenome.Rnorvegicus.UCSC.rn6-seed</li>
	<li>BSgenome.Scerevisiae.UCSC.sacCer1-seed</li>
	<li>BSgenome.Scerevisiae.UCSC.sacCer2-seed</li>
	<li>BSgenome.Scerevisiae.UCSC.sacCer3-seed</li>
	<li>BSgenome.Sscrofa.UCSC.susScr11-seed</li>
	<li>BSgenome.Sscrofa.UCSC.susScr11-tools/<ul>
	<li>sort_susScr11.2bit.R</li>
	</ul></li>
	<li>BSgenome.Sscrofa.UCSC.susScr3-seed</li>
	<li>BSgenome.Sscrofa.UCSC.susScr3-tools/<ul>
	<li>splitbigfasta.R</li>
	</ul></li>
	<li>BSgenome.Sscrofa.UCSC.susScr3.masked-seed</li>
	<li>BSgenome.Tguttata.UCSC.taeGut1-seed</li>
	<li>BSgenome.Tguttata.UCSC.taeGut1.masked-seed</li>
	<li>BSgenome.Tguttata.UCSC.taeGut2-seed</li>
	<li>BSgenome.Tguttata.UCSC.taeGut2-tools/<ul>
	<li>sort_taeGut2.2bit.R</li>
	</ul></li>
	<li>BSgenome.influenza.NCBI.20100628-seed</li>
	<li>ForgeAll.R</li>
	<li>bump_seed_version.sh</li>
	</ul></li>
	<li>ce2chrM.fa.gz</li>
	<li>ce2chrX.fa.gz</li>
	<li>ce2dict0.fa</li>
	</ul></li>
	<li>help/<ul>
	<li>+24+2CBSgenome-method.html</li>
	<li>+5B+2CXStringSet+2CGRangesList-method.html</li>
	<li>+5B+2CXStringSet+2CGenomicRanges-method.html</li>
	<li>+5B+5B+2CBSgenome-method.html</li>
	<li>AnIndex</li>
	<li>BSParams-class.html</li>
	<li>BSParams.html</li>
	<li>BSgenome-class.html</li>
	<li>BSgenome-utils.html</li>
	<li>BSgenome.html</li>
	<li>BSgenome.rdb</li>
	<li>BSgenome.rdx</li>
	<li>BSgenomeDataPkgSeed-class.html</li>
	<li>BSgenomeDataPkgSeed.html</li>
	<li>BSgenomeForge.html</li>
	<li>BSgenomeViews-class.html</li>
	<li>BSgenomeViews.html</li>
	<li>InjectSNPsHandler-class.html</li>
	<li>InjectSNPsHandler.html</li>
	<li>ODLT_SNPlocs-class.html</li>
	<li>ODLT_SNPlocs.html</li>
	<li>OldFashionSNPlocs-class.html</li>
	<li>OldFashionSNPlocs.html</li>
	<li>SNPlocs-class.html</li>
	<li>SNPlocs.html</li>
	<li>SNPlocs_pkgname+2CBSgenome-method.html</li>
	<li>SNPlocs_pkgname+2CInjectSNPsHandler-method.html</li>
	<li>SNPlocs_pkgname.html</li>
	<li>Views+2CBSgenome-method.html</li>
	<li>XtraSNPlocs-class.html</li>
	<li>XtraSNPlocs.html</li>
	<li>aliases.rds</li>
	<li>alphabetFrequency+2CBSgenomeViews-method.html</li>
	<li>as.character+2CBSgenomeViews-method.html</li>
	<li>as.data.frame+2CBSgenomeViews-method.html</li>
	<li>as.list+2CBSgenome-method.html</li>
	<li>available.SNPs.html</li>
	<li>available.genomes.html</li>
	<li>bsapply.html</li>
	<li>bsgenomeName+2CBSgenome-method.html</li>
	<li>class+3ABSParams.html</li>
	<li>class+3ABSgenome.html</li>
	<li>class+3ABSgenomeDataPkgSeed.html</li>
	<li>class+3ABSgenomeViews.html</li>
	<li>class+3AInjectSNPsHandler.html</li>
	<li>class+3AODLT_SNPlocs.html</li>
	<li>class+3AOldFashionSNPlocs.html</li>
	<li>class+3ASNPlocs.html</li>
	<li>class+3AXtraSNPlocs.html</li>
	<li>coerce+2CBSgenome+2CGenomeDescription-method.html</li>
	<li>coerce+2CBSgenomeViews+2CDNAStringSet-method.html</li>
	<li>coerce+2CBSgenomeViews+2CXStringSet-method.html</li>
	<li>colnames+2CXtraSNPlocs-method.html</li>
	<li>commonName+2CBSgenome-method.html</li>
	<li>commonName+2CSNPlocs-method.html</li>
	<li>commonName+2CXtraSNPlocs-method.html</li>
	<li>compatibleGenomes+2CSNPlocs-method.html</li>
	<li>compatibleGenomes.html</li>
	<li>consensusMatrix+2CBSgenomeViews-method.html</li>
	<li>consensusString+2CBSgenomeViews-method.html</li>
	<li>countPWM+2CBSgenome-method.html</li>
	<li>dim+2CXtraSNPlocs-method.html</li>
	<li>elementNROWS+2CBSgenomeViews-method.html</li>
	<li>end+2CBSgenomeViews-method.html</li>
	<li>export+2CBSgenome+2CFastaFile+2CANY-method.html</li>
	<li>export+2CBSgenome+2CTwoBitFile+2CANY-method.html</li>
	<li>export-methods.html</li>
	<li>extractROWS+2CBSgenomeViews-method.html</li>
	<li>forgeBSgenomeDataPkg+2CBSgenomeDataPkgSeed-method.html</li>
	<li>forgeBSgenomeDataPkg+2Ccharacter-method.html</li>
	<li>forgeBSgenomeDataPkg+2Clist-method.html</li>
	<li>forgeBSgenomeDataPkg.html</li>
	<li>forgeMasksFiles.html</li>
	<li>forgeSeqFiles.html</li>
	<li>forgeSeqlengthsRdaFile.html</li>
	<li>forgeSeqlengthsRdsFile.html</li>
	<li>getBSgenome.html</li>
	<li>getListElement+2CBSgenomeViews-method.html</li>
	<li>getSeq+2CBSgenome-method.html</li>
	<li>getSeq+2CXStringSet-method.html</li>
	<li>getSeq-methods.html</li>
	<li>getSeq.html</li>
	<li>granges+2CBSgenomeViews-method.html</li>
	<li>hasOnlyBaseLetters+2CBSgenomeViews-method.html</li>
	<li>inferRefAndAltAlleles.html</li>
	<li>injectSNPs+2CBSgenome-method.html</li>
	<li>injectSNPs.html</li>
	<li>installed.SNPs.html</li>
	<li>installed.genomes.html</li>
	<li>length+2CBSgenome-method.html</li>
	<li>length+2CBSgenomeViews-method.html</li>
	<li>letterFrequency+2CBSgenomeViews-method.html</li>
	<li>masknames+2CBSgenome-method.html</li>
	<li>masknames+2CMaskedBSgenome-method.html</li>
	<li>masknames.html</li>
	<li>matchPWM+2CBSgenome-method.html</li>
	<li>mseqnames+2CBSgenome-method.html</li>
	<li>mseqnames.html</li>
	<li>names+2CBSgenome-method.html</li>
	<li>names+2CBSgenomeViews-method.html</li>
	<li>nchar+2CBSgenomeViews-method.html</li>
	<li>newSNPlocs.html</li>
	<li>newXtraSNPlocs.html</li>
	<li>new_ODLT_SNPlocs.html</li>
	<li>nucleotideFrequencyAt+2CBSgenomeViews-method.html</li>
	<li>oligonucleotideFrequency+2CBSgenomeViews-method.html</li>
	<li>organism+2CBSgenome-method.html</li>
	<li>organism+2CSNPlocs-method.html</li>
	<li>organism+2CXtraSNPlocs-method.html</li>
	<li>paths.rds</li>
	<li>provider+2CBSgenome-method.html</li>
	<li>provider+2CSNPlocs-method.html</li>
	<li>provider+2CXtraSNPlocs-method.html</li>
	<li>providerVersion+2CBSgenome-method.html</li>
	<li>providerVersion+2CSNPlocs-method.html</li>
	<li>providerVersion+2CXtraSNPlocs-method.html</li>
	<li>ranges+2CBSgenomeViews-method.html</li>
	<li>referenceGenome+2CSNPlocs-method.html</li>
	<li>referenceGenome+2CXtraSNPlocs-method.html</li>
	<li>referenceGenome.html</li>
	<li>releaseDate+2CBSgenome-method.html</li>
	<li>releaseDate+2CSNPlocs-method.html</li>
	<li>releaseDate+2CXtraSNPlocs-method.html</li>
	<li>releaseName+2CBSgenome-method.html</li>
	<li>releaseName+2CSNPlocs-method.html</li>
	<li>releaseName+2CXtraSNPlocs-method.html</li>
	<li>seqinfo+2CBSgenome-method.html</li>
	<li>seqinfo+2CBSgenomeViews-method.html</li>
	<li>seqinfo+2CSNPlocs-method.html</li>
	<li>seqinfo+2CXtraSNPlocs-method.html</li>
	<li>seqinfo+3C-+2CBSgenome-method.html</li>
	<li>seqnames+2CBSgenome-method.html</li>
	<li>seqnames+2CBSgenomeViews-method.html</li>
	<li>seqnames+2CSNPlocs-method.html</li>
	<li>seqnames+2CXtraSNPlocs-method.html</li>
	<li>seqnames+3C-+2CBSgenome-method.html</li>
	<li>seqtype+2CBSgenomeViews-method.html</li>
	<li>show+2CBSgenome-method.html</li>
	<li>show+2CBSgenomeViews-method.html</li>
	<li>show+2CSNPlocs-method.html</li>
	<li>show+2CXtraSNPlocs-method.html</li>
	<li>snpcount+2CBSgenome-method.html</li>
	<li>snpcount+2CInjectSNPsHandler-method.html</li>
	<li>snpcount+2CODLT_SNPlocs-method.html</li>
	<li>snpcount+2COldFashionSNPlocs-method.html</li>
	<li>snpcount+2CSNPlocs-method.html</li>
	<li>snpcount+2CXtraSNPlocs-method.html</li>
	<li>snpcount.html</li>
	<li>snpid2alleles+2COldFashionSNPlocs-method.html</li>
	<li>snpid2alleles+2CSNPlocs-method.html</li>
	<li>snpid2alleles.html</li>
	<li>snpid2grange+2COldFashionSNPlocs-method.html</li>
	<li>snpid2grange+2CSNPlocs-method.html</li>
	<li>snpid2grange.html</li>
	<li>snpid2loc+2COldFashionSNPlocs-method.html</li>
	<li>snpid2loc+2CSNPlocs-method.html</li>
	<li>snpid2loc.html</li>
	<li>snplocs+2CBSgenome-method.html</li>
	<li>snplocs+2CInjectSNPsHandler-method.html</li>
	<li>snplocs+2CODLT_SNPlocs-method.html</li>
	<li>snplocs+2COldFashionSNPlocs-method.html</li>
	<li>snplocs+2CSNPlocs-method.html</li>
	<li>snplocs.html</li>
	<li>snpsById+2CODLT_SNPlocs-method.html</li>
	<li>snpsById+2COldFashionSNPlocs-method.html</li>
	<li>snpsById+2CSNPlocs-method.html</li>
	<li>snpsById+2CXtraSNPlocs-method.html</li>
	<li>snpsById.html</li>
	<li>snpsByOverlaps+2CODLT_SNPlocs-method.html</li>
	<li>snpsByOverlaps+2COldFashionSNPlocs-method.html</li>
	<li>snpsByOverlaps+2CSNPlocs-method.html</li>
	<li>snpsByOverlaps+2CXtraSNPlocs-method.html</li>
	<li>snpsByOverlaps.html</li>
	<li>snpsBySeqname+2CODLT_SNPlocs-method.html</li>
	<li>snpsBySeqname+2COldFashionSNPlocs-method.html</li>
	<li>snpsBySeqname+2CSNPlocs-method.html</li>
	<li>snpsBySeqname+2CXtraSNPlocs-method.html</li>
	<li>snpsBySeqname.html</li>
	<li>sourceUrl+2CBSgenome-method.html</li>
	<li>sourceUrl.html</li>
	<li>species+2CXtraSNPlocs-method.html</li>
	<li>start+2CBSgenomeViews-method.html</li>
	<li>strand+2CBSgenomeViews-method.html</li>
	<li>subject+2CBSgenomeViews-method.html</li>
	<li>uniqueLetters+2CBSgenomeViews-method.html</li>
	<li>unlist+2CBSgenomeViews-method.html</li>
	<li>vcountPDict+2CBSgenome-method.html</li>
	<li>vcountPattern+2CBSgenome-method.html</li>
	<li>vmatchPDict+2CBSgenome-method.html</li>
	<li>vmatchPattern+2CBSgenome-method.html</li>
	<li>width+2CBSgenomeViews-method.html</li>
	<li>writeBSgenomeToFasta.html</li>
	<li>writeBSgenomeToTwobit.html</li>
	</ul></li>
	<li>html/<ul>
	<li>00Index.html</li>
	<li>BSParams-class.html</li>
	<li>BSgenome-class.html</li>
	<li>BSgenome-utils.html</li>
	<li>BSgenomeForge.html</li>
	<li>BSgenomeViews-class.html</li>
	<li>R.css</li>
	<li>SNPlocs-class.html</li>
	<li>XtraSNPlocs-class.html</li>
	<li>available.genomes.html</li>
	<li>bsapply.html</li>
	<li>export-methods.html</li>
	<li>getSeq-methods.html</li>
	<li>injectSNPs.html</li>
	</ul></li>
	<li>pkgtemplates/<ul>
	<li>BSgenome_datapkg/<ul>
	<li>DESCRIPTION</li>
	<li>NAMESPACE</li>
	<li>R/<ul>
	<li>zzz.R</li>
	</ul></li>
	<li>inst/<ul>
	<li>extdata/<ul>
	</ul></li>
	</ul></li>
	<li>man/<ul>
	<li>package.Rd</li>
	</ul></li>
	</ul></li>
	<li>MaskedBSgenome_datapkg/<ul>
	<li>DESCRIPTION</li>
	<li>NAMESPACE</li>
	<li>R/<ul>
	<li>zzz.R</li>
	</ul></li>
	<li>inst/<ul>
	<li>extdata/<ul>
	</ul></li>
	</ul></li>
	<li>man/<ul>
	<li>package.Rd</li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
</div>

  <div class="text-muted text-center">
    <br>
    Sources on <a href="https://pagure.io/fedora-packages-static">Pagure</a>
  </div>
</div>