<!DOCTYPE html>

<title>R-Biostrings-2.64.1-1.fc37 - Fedora Packages</title>
<link rel="stylesheet" href="../../../assets/css/bootstrap.min.css">
<link rel="stylesheet" href="../../../assets/css/main.css">
<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta charset="UTF-8">
<meta name="description" content="View R-Biostrings-2.64.1-1.fc37 in Fedora 37. R-Biostrings: String objects representing biological sequences">

<div class="container">
  <div class="row">
    <a href="../../../" class="col-md-6 d-flex align-items-center justify-content-center justify-content-md-start p-1">
      <img alt="Fedora Packages" src="../../../assets/images/logo.png">
    </a>
    <div class="col-md-6 d-flex align-items-center">
    </div>
  </div>

  <hr />

<h1>
	R-Biostrings-2.64.1-1.fc37<small class="text-muted"> in Fedora 37</small>
</h1>
<p>
	<a href=".">&crarr; Return to the main page of R-Biostrings</a><br>
	<a href="https://koji.fedoraproject.org/koji/search?match=exact&type=build&terms=R-Biostrings-2.64.1-1.fc37">View build</a><br>
	<a href="https://bodhi.fedoraproject.org/updates/?search=R-Biostrings-2.64.1-1.fc37">Search for updates</a>
</p>

<p>
	<b>Package Info <span class="text-muted">(Data from x86_64 build)</span></b><br>
	<a href="#changelog">&#129047; Changelog</a><br>
	<a href="#dependencies">&#129047; Dependencies</a><br>
	<a href="#provides">&#129047; Provides</a><br>
	<a href="#files">&#129047; Files</a><br>
</p>

<h2 id="changelog">Changelog</h2>
<div class="table-responsive">
	<table class="table table-striped table-borderless ">
		<thead>
			<tr>
				<th scope="col">Date</th>
				<th scope="col">Author</th>
				<th scope="col">Change</th>
			</tr>
		</thead>
		<tr>
			<td>2022-09-03</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.64.1-1</td>
			<td>- update to 2.64.1
- rebuild for R 4.2.1</td>
		</tr>
		<tr>
			<td>2022-07-20</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.60.1-4</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_37_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2022-01-19</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.60.1-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_36_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-07-21</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.60.1-2</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_35_Mass_Rebuild</td>
		</tr>
		<tr>
			<td>2021-06-16</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.60.1-1</td>
			<td>- update to 2.60.1
- Rebuilt for R 4.1.0</td>
		</tr>
		<tr>
			<td>2021-02-03</td>
			<td>Tom Callaway &lt;spot at fedoraproject dot org&gt; - 2.58.0-1</td>
			<td>- update to 2.58.0</td>
		</tr>
		<tr>
			<td>2021-01-25</td>
			<td>Fedora Release Engineering &lt;releng at fedoraproject dot org&gt; - 2.56.0-3</td>
			<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
		</tr>
		<tbody>
		</tbody>
	</table>
</div>

<div class="row">
	<div class="col">
		<h2 id="dependencies">Dependencies</h2>
		<ul>
			<li>
			<a href="/pkgs/R-BiocGenerics/R-BiocGenerics/fedora-37.html">
			R-BiocGenerics</a> &gt;= 0.37.0

			</li>
			<li>
			<a href="/pkgs/R-GenomeInfoDb/R-GenomeInfoDb/fedora-37.html">
			R-GenomeInfoDb</a>

			</li>
			<li>
			<a href="/pkgs/R-IRanges/R-IRanges/fedora-37.html">
			R-IRanges</a> &gt;= 2.30.1

			</li>
			<li>
			<a href="/pkgs/R-S4Vectors/R-S4Vectors/fedora-37.html">
			R-S4Vectors</a> &gt;= 0.27.12

			</li>
			<li>
			<a href="/pkgs/R-XVector/R-XVector/fedora-37.html">
			R-XVector</a> &gt;= 0.29.2

			</li>
			<li>
			<a href="/pkgs/R/R-core/fedora-37.html">
			R-core</a> = 4.2

			</li>
			<li>
			<a href="/pkgs/R-crayon/R-crayon/fedora-37.html">
			R-crayon</a>

			</li>
			<li>
			<a href="/pkgs/glibc/glibc/fedora-37.html">
			glibc</a>

			</li>
		</ul>
	</div>
	<div class="col">
		<h2 id="provides">Provides</h2>
		<ul>
			<li>R(Biostrings)</li>
			<li>R-Biostrings</li>
			<li>R-Biostrings(x86-64)</li>
		</ul>
	</div>
</div>

<h2 id="files">Files</h2>

<div class="tree">
	<ul>
	<li>usr/<ul>
	<li>lib/<ul>
	<li>.build-id/<ul>
	<li>73/<ul>
	<li>f0857ea8bf54a816e70e3c2fe7f7c725fefb66</li>
	</ul></li>
	</ul></li>
	</ul></li>
	<li>lib64/<ul>
	<li>R/<ul>
	<li>library/<ul>
	<li>Biostrings/<ul>
	<li>DESCRIPTION</li>
	<li>INDEX</li>
	<li>Meta/<ul>
	<li>Rd.rds</li>
	<li>data.rds</li>
	<li>features.rds</li>
	<li>hsearch.rds</li>
	<li>links.rds</li>
	<li>nsInfo.rds</li>
	<li>package.rds</li>
	<li>vignette.rds</li>
	</ul></li>
	<li>NAMESPACE</li>
	<li>NEWS</li>
	<li>R/<ul>
	<li>Biostrings</li>
	<li>Biostrings.rdb</li>
	<li>Biostrings.rdx</li>
	</ul></li>
	<li>data/<ul>
	<li>BLOSUM100.rda</li>
	<li>BLOSUM45.rda</li>
	<li>BLOSUM50.rda</li>
	<li>BLOSUM62.rda</li>
	<li>BLOSUM80.rda</li>
	<li>HNF4alpha.rda</li>
	<li>PAM120.rda</li>
	<li>PAM250.rda</li>
	<li>PAM30.rda</li>
	<li>PAM40.rda</li>
	<li>PAM70.rda</li>
	<li>phiX174Phage.rda</li>
	<li>srPhiX174.rda</li>
	<li>yeastSEQCHR1.rda</li>
	</ul></li>
	<li>doc/<ul>
	<li>Biostrings2Classes.R</li>
	<li>Biostrings2Classes.Rnw</li>
	<li>Biostrings2Classes.pdf</li>
	<li>BiostringsQuickOverview.Rnw</li>
	<li>BiostringsQuickOverview.pdf</li>
	<li>MultipleAlignments.R</li>
	<li>MultipleAlignments.Rnw</li>
	<li>MultipleAlignments.pdf</li>
	<li>PairwiseAlignments.R</li>
	<li>PairwiseAlignments.Rnw</li>
	<li>PairwiseAlignments.pdf</li>
	<li>index.html</li>
	<li>matchprobes.R</li>
	<li>matchprobes.Rnw</li>
	<li>matchprobes.pdf</li>
	</ul></li>
	<li>extdata/<ul>
	<li>Phylip.txt</li>
	<li>README</li>
	<li>Sc.fa</li>
	<li>Sp.fa</li>
	<li>dm3_upstream2000.fa.gz</li>
	<li>fastaEx.fa</li>
	<li>hg18chr8_gap.txt</li>
	<li>hg18chrY_gap.txt</li>
	<li>msx2_mRNA.aln</li>
	<li>s_1_sequence.txt</li>
	<li>someORF.fa</li>
	<li>someORF.fa.gz</li>
	</ul></li>
	<li>help/<ul>
	<li>+21+3D+2CBString+2Ccharacter-method.html</li>
	<li>+21+3D+2CXString+2CXString-method.html</li>
	<li>+21+3D+2CXString+2CXStringViews-method.html</li>
	<li>+21+3D+2CXStringViews+2CXString-method.html</li>
	<li>+21+3D+2CXStringViews+2CXStringViews-method.html</li>
	<li>+21+3D+2CXStringViews+2Ccharacter-method.html</li>
	<li>+21+3D+2Ccharacter+2CBString-method.html</li>
	<li>+21+3D+2Ccharacter+2CXStringViews-method.html</li>
	<li>+3D+3D+2CBString+2Ccharacter-method.html</li>
	<li>+3D+3D+2CXString+2CXString-method.html</li>
	<li>+3D+3D+2CXString+2CXStringViews-method.html</li>
	<li>+3D+3D+2CXStringViews+2CXString-method.html</li>
	<li>+3D+3D+2CXStringViews+2CXStringViews-method.html</li>
	<li>+3D+3D+2CXStringViews+2Ccharacter-method.html</li>
	<li>+3D+3D+2Ccharacter+2CBString-method.html</li>
	<li>+3D+3D+2Ccharacter+2CXStringViews-method.html</li>
	<li>+5B+2CXStringPartialMatches-method.html</li>
	<li>+5B+5B+2CByPos_MIndex-method.html</li>
	<li>+5B+5B+2CPDict-method.html</li>
	<li>+5B+5B+2CSparseList-method.html</li>
	<li>.inplaceReplaceLetterAt.html</li>
	<li>AAMultipleAlignment-class.html</li>
	<li>AAMultipleAlignment.html</li>
	<li>AAString-class.html</li>
	<li>AAString.html</li>
	<li>AAStringSet-class.html</li>
	<li>AAStringSet.html</li>
	<li>AAStringSetList-class.html</li>
	<li>AAStringSetList.html</li>
	<li>AA_ALPHABET.html</li>
	<li>AA_PROTEINOGENIC.html</li>
	<li>AA_STANDARD.html</li>
	<li>ACtree2-class.html</li>
	<li>ACtree2.html</li>
	<li>AMINO_ACID_CODE.html</li>
	<li>AlignedXStringSet-class.html</li>
	<li>AlignedXStringSet.html</li>
	<li>AlignedXStringSet0-class.html</li>
	<li>AlignedXStringSet0.html</li>
	<li>AnIndex</li>
	<li>BLOSUM100.html</li>
	<li>BLOSUM45.html</li>
	<li>BLOSUM50.html</li>
	<li>BLOSUM62.html</li>
	<li>BLOSUM80.html</li>
	<li>BString-class.html</li>
	<li>BString.html</li>
	<li>BStringSet-class.html</li>
	<li>BStringSet.html</li>
	<li>BStringSetList-class.html</li>
	<li>BStringSetList.html</li>
	<li>Biostrings-internals.html</li>
	<li>Biostrings.rdb</li>
	<li>Biostrings.rdx</li>
	<li>ByPos_MIndex-class.html</li>
	<li>DNAMultipleAlignment-class.html</li>
	<li>DNAMultipleAlignment.html</li>
	<li>DNAString-class.html</li>
	<li>DNAString.html</li>
	<li>DNAStringSet-class.html</li>
	<li>DNAStringSet.html</li>
	<li>DNAStringSetList-class.html</li>
	<li>DNAStringSetList.html</li>
	<li>DNA_ALPHABET.html</li>
	<li>DNA_BASES.html</li>
	<li>Expanded_TB_PDict-class.html</li>
	<li>Expanded_TB_PDict.html</li>
	<li>GENETIC_CODE.html</li>
	<li>GENETIC_CODE_TABLE.html</li>
	<li>HNF4alpha.html</li>
	<li>IUPAC_CODE_MAP.html</li>
	<li>IlluminaQuality-class.html</li>
	<li>IlluminaQuality.html</li>
	<li>InDel-class.html</li>
	<li>InDel.html</li>
	<li>MIndex-class.html</li>
	<li>MIndex.html</li>
	<li>MTB_PDict-class.html</li>
	<li>MTB_PDict.html</li>
	<li>MaskedAAString-class.html</li>
	<li>MaskedAAString.html</li>
	<li>MaskedBString-class.html</li>
	<li>MaskedBString.html</li>
	<li>MaskedDNAString-class.html</li>
	<li>MaskedDNAString.html</li>
	<li>MaskedRNAString-class.html</li>
	<li>MaskedRNAString.html</li>
	<li>MaskedXString-class.html</li>
	<li>MaskedXString.html</li>
	<li>MultipleAlignment-class.html</li>
	<li>MultipleAlignment.html</li>
	<li>N50.html</li>
	<li>PAM120.html</li>
	<li>PAM250.html</li>
	<li>PAM30.html</li>
	<li>PAM40.html</li>
	<li>PAM70.html</li>
	<li>PDict+2CAsIs-method.html</li>
	<li>PDict+2CDNAStringSet-method.html</li>
	<li>PDict+2CXStringViews-method.html</li>
	<li>PDict+2Ccharacter-method.html</li>
	<li>PDict+2Cprobetable-method.html</li>
	<li>PDict-class.html</li>
	<li>PDict.html</li>
	<li>PDict3Parts-class.html</li>
	<li>PDict3Parts.html</li>
	<li>PWM+2CDNAStringSet-method.html</li>
	<li>PWM+2Ccharacter-method.html</li>
	<li>PWM+2Cmatrix-method.html</li>
	<li>PWM.html</li>
	<li>PWMscoreStartingAt.html</li>
	<li>PairwiseAlignments+2CXString+2CXString-method.html</li>
	<li>PairwiseAlignments+2CXStringSet+2Cmissing-method.html</li>
	<li>PairwiseAlignments+2Ccharacter+2Ccharacter-method.html</li>
	<li>PairwiseAlignments+2Ccharacter+2Cmissing-method.html</li>
	<li>PairwiseAlignments-class.html</li>
	<li>PairwiseAlignments-io.html</li>
	<li>PairwiseAlignments.html</li>
	<li>PairwiseAlignmentsSingleSubject+2CXString+2CXString-method.html</li>
	<li>PairwiseAlignmentsSingleSubject+2CXStringSet+2Cmissing-method.html</li>
	<li>PairwiseAlignmentsSingleSubject+2Ccharacter+2Ccharacter-method.html</li>
	<li>PairwiseAlignmentsSingleSubject+2Ccharacter+2Cmissing-method.html</li>
	<li>PairwiseAlignmentsSingleSubject-class.html</li>
	<li>PairwiseAlignmentsSingleSubject.html</li>
	<li>PairwiseAlignmentsSingleSubjectSummary-class.html</li>
	<li>PairwiseAlignmentsSingleSubjectSummary.html</li>
	<li>PhredQuality-class.html</li>
	<li>PhredQuality.html</li>
	<li>PreprocessedTB-class.html</li>
	<li>PreprocessedTB.html</li>
	<li>QualityAlignedXStringSet-class.html</li>
	<li>QualityAlignedXStringSet.html</li>
	<li>QualityScaledAAStringSet-class.html</li>
	<li>QualityScaledAAStringSet.html</li>
	<li>QualityScaledBStringSet-class.html</li>
	<li>QualityScaledBStringSet.html</li>
	<li>QualityScaledDNAStringSet-class.html</li>
	<li>QualityScaledDNAStringSet.html</li>
	<li>QualityScaledRNAStringSet-class.html</li>
	<li>QualityScaledRNAStringSet.html</li>
	<li>QualityScaledXStringSet-class.html</li>
	<li>QualityScaledXStringSet.html</li>
	<li>RNAMultipleAlignment-class.html</li>
	<li>RNAMultipleAlignment.html</li>
	<li>RNAString-class.html</li>
	<li>RNAString.html</li>
	<li>RNAStringSet-class.html</li>
	<li>RNAStringSet.html</li>
	<li>RNAStringSetList-class.html</li>
	<li>RNAStringSetList.html</li>
	<li>RNA_ALPHABET.html</li>
	<li>RNA_BASES.html</li>
	<li>RNA_GENETIC_CODE.html</li>
	<li>SolexaQuality-class.html</li>
	<li>SolexaQuality.html</li>
	<li>SparseList-class.html</li>
	<li>SparseList.html</li>
	<li>TB_PDict-class.html</li>
	<li>TB_PDict.html</li>
	<li>Twobit-class.html</li>
	<li>Twobit.html</li>
	<li>Views+2CMaskedXString-method.html</li>
	<li>Views+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>Views+2CXString-method.html</li>
	<li>Views+2Ccharacter-method.html</li>
	<li>XString-class.html</li>
	<li>XString.html</li>
	<li>XStringCodec-class.html</li>
	<li>XStringCodec.html</li>
	<li>XStringPartialMatches-class.html</li>
	<li>XStringQuality-class.html</li>
	<li>XStringQuality.html</li>
	<li>XStringSet-class.html</li>
	<li>XStringSet-comparison.html</li>
	<li>XStringSet-io.html</li>
	<li>XStringSet.html</li>
	<li>XStringSetList-class.html</li>
	<li>XStringSetList.html</li>
	<li>XStringViews-class.html</li>
	<li>XStringViews.html</li>
	<li>aliases.rds</li>
	<li>align-utils.html</li>
	<li>aligned+2CAlignedXStringSet0-method.html</li>
	<li>aligned+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>aligned.html</li>
	<li>alignedPattern+2CPairwiseAlignments-method.html</li>
	<li>alignedPattern.html</li>
	<li>alignedSubject+2CPairwiseAlignments-method.html</li>
	<li>alignedSubject.html</li>
	<li>alphabet+2CANY-method.html</li>
	<li>alphabet+2CXStringQuality-method.html</li>
	<li>alphabet.html</li>
	<li>alphabetFrequency+2CAAString-method.html</li>
	<li>alphabetFrequency+2CAAStringSet-method.html</li>
	<li>alphabetFrequency+2CDNAString-method.html</li>
	<li>alphabetFrequency+2CDNAStringSet-method.html</li>
	<li>alphabetFrequency+2CMaskedXString-method.html</li>
	<li>alphabetFrequency+2CMultipleAlignment-method.html</li>
	<li>alphabetFrequency+2CRNAString-method.html</li>
	<li>alphabetFrequency+2CRNAStringSet-method.html</li>
	<li>alphabetFrequency+2CXString-method.html</li>
	<li>alphabetFrequency+2CXStringSet-method.html</li>
	<li>alphabetFrequency+2CXStringViews-method.html</li>
	<li>alphabetFrequency.html</li>
	<li>anyNA+2CXStringSet-method.html</li>
	<li>as.character+2CAlignedXStringSet0-method.html</li>
	<li>as.character+2CMaskedXString-method.html</li>
	<li>as.character+2CMultipleAlignment-method.html</li>
	<li>as.character+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>as.character+2CXString-method.html</li>
	<li>as.character+2CXStringSet-method.html</li>
	<li>as.character+2CXStringViews-method.html</li>
	<li>as.data.frame+2CXStringSet-method.html</li>
	<li>as.data.frame+2CXStringViews-method.html</li>
	<li>as.factor+2CXStringSet-method.html</li>
	<li>as.list+2CMTB_PDict-method.html</li>
	<li>as.list+2CSparseList-method.html</li>
	<li>as.matrix+2CMultipleAlignment-method.html</li>
	<li>as.matrix+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>as.matrix+2CXStringQuality-method.html</li>
	<li>as.matrix+2CXStringSet-method.html</li>
	<li>as.matrix+2CXStringViews-method.html</li>
	<li>as.vector+2CXString-method.html</li>
	<li>as.vector+2CXStringQuality-method.html</li>
	<li>as.vector+2CXStringSet-method.html</li>
	<li>chartr+2CANY+2CANY+2CMaskedXString-method.html</li>
	<li>chartr+2CANY+2CANY+2CXString-method.html</li>
	<li>chartr+2CANY+2CANY+2CXStringSet-method.html</li>
	<li>chartr+2CANY+2CANY+2CXStringViews-method.html</li>
	<li>chartr.html</li>
	<li>class+3AAAMultipleAlignment.html</li>
	<li>class+3AAAString.html</li>
	<li>class+3AAAStringSet.html</li>
	<li>class+3AAAStringSetList.html</li>
	<li>class+3AACtree2.html</li>
	<li>class+3AAlignedXStringSet.html</li>
	<li>class+3AAlignedXStringSet0.html</li>
	<li>class+3ABString.html</li>
	<li>class+3ABStringSet.html</li>
	<li>class+3ABStringSetList.html</li>
	<li>class+3AByPos_MIndex.html</li>
	<li>class+3ADNAMultipleAlignment.html</li>
	<li>class+3ADNAString.html</li>
	<li>class+3ADNAStringSet.html</li>
	<li>class+3ADNAStringSetList.html</li>
	<li>class+3AExpanded_TB_PDict.html</li>
	<li>class+3AIlluminaQuality.html</li>
	<li>class+3AInDel.html</li>
	<li>class+3AMIndex.html</li>
	<li>class+3AMTB_PDict.html</li>
	<li>class+3AMaskedAAString.html</li>
	<li>class+3AMaskedBString.html</li>
	<li>class+3AMaskedDNAString.html</li>
	<li>class+3AMaskedRNAString.html</li>
	<li>class+3AMaskedXString.html</li>
	<li>class+3AMultipleAlignment.html</li>
	<li>class+3APDict.html</li>
	<li>class+3APDict3Parts.html</li>
	<li>class+3APairwiseAlignments.html</li>
	<li>class+3APairwiseAlignmentsSingleSubject.html</li>
	<li>class+3APairwiseAlignmentsSingleSubjectSummary.html</li>
	<li>class+3APhredQuality.html</li>
	<li>class+3APreprocessedTB.html</li>
	<li>class+3AQualityAlignedXStringSet.html</li>
	<li>class+3AQualityScaledAAStringSet.html</li>
	<li>class+3AQualityScaledBStringSet.html</li>
	<li>class+3AQualityScaledDNAStringSet.html</li>
	<li>class+3AQualityScaledRNAStringSet.html</li>
	<li>class+3AQualityScaledXStringSet.html</li>
	<li>class+3ARNAMultipleAlignment.html</li>
	<li>class+3ARNAString.html</li>
	<li>class+3ARNAStringSet.html</li>
	<li>class+3ARNAStringSetList.html</li>
	<li>class+3ASolexaQuality.html</li>
	<li>class+3ASparseList.html</li>
	<li>class+3ATB_PDict.html</li>
	<li>class+3ATwobit.html</li>
	<li>class+3AXString.html</li>
	<li>class+3AXStringCodec.html</li>
	<li>class+3AXStringPartialMatches.html</li>
	<li>class+3AXStringQuality.html</li>
	<li>class+3AXStringSet.html</li>
	<li>class+3AXStringSetList.html</li>
	<li>class+3AXStringViews.html</li>
	<li>codons+2CDNAString-method.html</li>
	<li>codons+2CMaskedDNAString-method.html</li>
	<li>codons+2CMaskedRNAString-method.html</li>
	<li>codons+2CRNAString-method.html</li>
	<li>codons.html</li>
	<li>coerce+2CAAString+2CMaskedAAString-method.html</li>
	<li>coerce+2CANY+2CAAStringSet-method.html</li>
	<li>coerce+2CANY+2CBStringSet-method.html</li>
	<li>coerce+2CANY+2CDNAStringSet-method.html</li>
	<li>coerce+2CANY+2CRNAStringSet-method.html</li>
	<li>coerce+2CANY+2CXStringSet-method.html</li>
	<li>coerce+2CBString+2CIlluminaQuality-method.html</li>
	<li>coerce+2CBString+2CMaskedBString-method.html</li>
	<li>coerce+2CBString+2CPhredQuality-method.html</li>
	<li>coerce+2CBString+2CSolexaQuality-method.html</li>
	<li>coerce+2CBStringSet+2CIlluminaQuality-method.html</li>
	<li>coerce+2CBStringSet+2CPhredQuality-method.html</li>
	<li>coerce+2CBStringSet+2CSolexaQuality-method.html</li>
	<li>coerce+2CDNAString+2CMaskedDNAString-method.html</li>
	<li>coerce+2CIntegerList+2CIlluminaQuality-method.html</li>
	<li>coerce+2CIntegerList+2CPhredQuality-method.html</li>
	<li>coerce+2CIntegerList+2CSolexaQuality-method.html</li>
	<li>coerce+2CList+2CAAStringSetList-method.html</li>
	<li>coerce+2CList+2CBStringSetList-method.html</li>
	<li>coerce+2CList+2CDNAStringSetList-method.html</li>
	<li>coerce+2CList+2CRNAStringSetList-method.html</li>
	<li>coerce+2CList+2CXStringSetList-method.html</li>
	<li>coerce+2CMIndex+2CCompressedIRangesList-method.html</li>
	<li>coerce+2CMaskedAAString+2CAAString-method.html</li>
	<li>coerce+2CMaskedBString+2CBString-method.html</li>
	<li>coerce+2CMaskedDNAString+2CDNAString-method.html</li>
	<li>coerce+2CMaskedRNAString+2CRNAString-method.html</li>
	<li>coerce+2CMaskedXString+2CMaskCollection-method.html</li>
	<li>coerce+2CMaskedXString+2CMaskedAAString-method.html</li>
	<li>coerce+2CMaskedXString+2CMaskedBString-method.html</li>
	<li>coerce+2CMaskedXString+2CMaskedDNAString-method.html</li>
	<li>coerce+2CMaskedXString+2CMaskedRNAString-method.html</li>
	<li>coerce+2CMaskedXString+2CNormalIRanges-method.html</li>
	<li>coerce+2CMaskedXString+2CViews-method.html</li>
	<li>coerce+2CMaskedXString+2CXStringViews-method.html</li>
	<li>coerce+2CMultipleAlignment+2CAAStringSet-method.html</li>
	<li>coerce+2CMultipleAlignment+2CBStringSet-method.html</li>
	<li>coerce+2CMultipleAlignment+2CDNAStringSet-method.html</li>
	<li>coerce+2CMultipleAlignment+2CRNAStringSet-method.html</li>
	<li>coerce+2CNumericList+2CIlluminaQuality-method.html</li>
	<li>coerce+2CNumericList+2CPhredQuality-method.html</li>
	<li>coerce+2CNumericList+2CSolexaQuality-method.html</li>
	<li>coerce+2CRNAString+2CMaskedRNAString-method.html</li>
	<li>coerce+2CXString+2CAAString-method.html</li>
	<li>coerce+2CXString+2CBString-method.html</li>
	<li>coerce+2CXString+2CDNAString-method.html</li>
	<li>coerce+2CXString+2CRNAString-method.html</li>
	<li>coerce+2CXStringQuality+2CCompressedIntegerList-method.html</li>
	<li>coerce+2CXStringQuality+2CCompressedNumericList-method.html</li>
	<li>coerce+2CXStringQuality+2CIntegerList-method.html</li>
	<li>coerce+2CXStringQuality+2CNumericList-method.html</li>
	<li>coerce+2CXStringQuality+2Cmatrix-method.html</li>
	<li>coerce+2CXStringSet+2CViews-method.html</li>
	<li>coerce+2CXStringSet+2CXStringViews-method.html</li>
	<li>coerce+2CXStringViews+2CAAStringSet-method.html</li>
	<li>coerce+2CXStringViews+2CBStringSet-method.html</li>
	<li>coerce+2CXStringViews+2CDNAStringSet-method.html</li>
	<li>coerce+2CXStringViews+2CRNAStringSet-method.html</li>
	<li>coerce+2CXStringViews+2CXStringSet-method.html</li>
	<li>coerce+2Ccharacter+2CAAMultipleAlignment-method.html</li>
	<li>coerce+2Ccharacter+2CAAString-method.html</li>
	<li>coerce+2Ccharacter+2CBString-method.html</li>
	<li>coerce+2Ccharacter+2CDNAMultipleAlignment-method.html</li>
	<li>coerce+2Ccharacter+2CDNAString-method.html</li>
	<li>coerce+2Ccharacter+2CIlluminaQuality-method.html</li>
	<li>coerce+2Ccharacter+2CPhredQuality-method.html</li>
	<li>coerce+2Ccharacter+2CRNAMultipleAlignment-method.html</li>
	<li>coerce+2Ccharacter+2CRNAString-method.html</li>
	<li>coerce+2Ccharacter+2CSolexaQuality-method.html</li>
	<li>coerce+2Ccharacter+2CXString-method.html</li>
	<li>coerce+2Cinteger+2CIlluminaQuality-method.html</li>
	<li>coerce+2Cinteger+2CPhredQuality-method.html</li>
	<li>coerce+2Cinteger+2CSolexaQuality-method.html</li>
	<li>coerce+2Clist+2CAAStringSetList-method.html</li>
	<li>coerce+2Clist+2CBStringSetList-method.html</li>
	<li>coerce+2Clist+2CDNAStringSetList-method.html</li>
	<li>coerce+2Clist+2CRNAStringSetList-method.html</li>
	<li>coerce+2Clist+2CXStringSetList-method.html</li>
	<li>coerce+2Cnumeric+2CIlluminaQuality-method.html</li>
	<li>coerce+2Cnumeric+2CPhredQuality-method.html</li>
	<li>coerce+2Cnumeric+2CSolexaQuality-method.html</li>
	<li>collapse+2CMaskedXString-method.html</li>
	<li>colmask+2CMultipleAlignment-method.html</li>
	<li>colmask+3C-+2CMultipleAlignment+2CANY-method.html</li>
	<li>colmask+3C-+2CMultipleAlignment+2CNULL-method.html</li>
	<li>colmask+3C-.html</li>
	<li>colmask.html</li>
	<li>compareStrings+2CAlignedXStringSet0+2CAlignedXStringSet0-method.html</li>
	<li>compareStrings+2CPairwiseAlignments+2Cmissing-method.html</li>
	<li>compareStrings+2CXString+2CXString-method.html</li>
	<li>compareStrings+2CXStringSet+2CXStringSet-method.html</li>
	<li>compareStrings+2Ccharacter+2Ccharacter-method.html</li>
	<li>compareStrings.html</li>
	<li>complement+2CDNAString-method.html</li>
	<li>complement+2CDNAStringSet-method.html</li>
	<li>complement+2CMaskedDNAString-method.html</li>
	<li>complement+2CMaskedRNAString-method.html</li>
	<li>complement+2CRNAString-method.html</li>
	<li>complement+2CRNAStringSet-method.html</li>
	<li>complement+2CXStringViews-method.html</li>
	<li>complement.html</li>
	<li>computeAllFlinks+2CACtree2-method.html</li>
	<li>computeAllFlinks.html</li>
	<li>consensusMatrix+2CMultipleAlignment-method.html</li>
	<li>consensusMatrix+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>consensusMatrix+2CXStringSet-method.html</li>
	<li>consensusMatrix+2CXStringViews-method.html</li>
	<li>consensusMatrix+2Ccharacter-method.html</li>
	<li>consensusMatrix+2Cmatrix-method.html</li>
	<li>consensusMatrix.html</li>
	<li>consensusString+2CAAMultipleAlignment-method.html</li>
	<li>consensusString+2CANY-method.html</li>
	<li>consensusString+2CBStringSet-method.html</li>
	<li>consensusString+2CDNAMultipleAlignment-method.html</li>
	<li>consensusString+2CDNAStringSet-method.html</li>
	<li>consensusString+2CMultipleAlignment-method.html</li>
	<li>consensusString+2CRNAMultipleAlignment-method.html</li>
	<li>consensusString+2CRNAStringSet-method.html</li>
	<li>consensusString+2CXStringViews-method.html</li>
	<li>consensusString+2Cmatrix-method.html</li>
	<li>consensusString.html</li>
	<li>consensusViews+2CAAMultipleAlignment-method.html</li>
	<li>consensusViews+2CDNAMultipleAlignment-method.html</li>
	<li>consensusViews+2CMultipleAlignment-method.html</li>
	<li>consensusViews+2CRNAMultipleAlignment-method.html</li>
	<li>consensusViews.html</li>
	<li>countPDict+2CMaskedXString-method.html</li>
	<li>countPDict+2CXString-method.html</li>
	<li>countPDict+2CXStringSet-method.html</li>
	<li>countPDict+2CXStringViews-method.html</li>
	<li>countPDict.html</li>
	<li>countPWM+2CDNAString-method.html</li>
	<li>countPWM+2CMaskedDNAString-method.html</li>
	<li>countPWM+2CXStringViews-method.html</li>
	<li>countPWM+2Ccharacter-method.html</li>
	<li>countPWM.html</li>
	<li>countPattern+2CMaskedXString-method.html</li>
	<li>countPattern+2CXString-method.html</li>
	<li>countPattern+2CXStringSet-method.html</li>
	<li>countPattern+2CXStringViews-method.html</li>
	<li>countPattern+2Ccharacter-method.html</li>
	<li>countPattern.html</li>
	<li>coverage+2CAlignedXStringSet0-method.html</li>
	<li>coverage+2CMIndex-method.html</li>
	<li>coverage+2CMaskedXString-method.html</li>
	<li>coverage+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>coverage+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>deletion+2CInDel-method.html</li>
	<li>deletion+2CPairwiseAlignments-method.html</li>
	<li>deletion.html</li>
	<li>detail+2CMultipleAlignment-method.html</li>
	<li>detail.html</li>
	<li>dim+2CMultipleAlignment-method.html</li>
	<li>dinucleotideFrequency.html</li>
	<li>dinucleotideFrequencyTest+2CDNAStringSet-method.html</li>
	<li>dinucleotideFrequencyTest+2CRNAStringSet-method.html</li>
	<li>dinucleotideFrequencyTest.html</li>
	<li>duplicated+2CPDict-method.html</li>
	<li>duplicated+2CPreprocessedTB-method.html</li>
	<li>elementNROWS+2CMIndex-method.html</li>
	<li>encoding+2CXStringQuality-method.html</li>
	<li>encoding.html</li>
	<li>end+2CAlignedXStringSet0-method.html</li>
	<li>endIndex+2CByPos_MIndex-method.html</li>
	<li>endIndex.html</li>
	<li>errorSubstitutionMatrices.html</li>
	<li>extractAllMatches.html</li>
	<li>extractAt+2CXString-method.html</li>
	<li>extractAt+2CXStringSet-method.html</li>
	<li>extractAt.html</li>
	<li>extract_character_from_XString_by_positions+2CXString-method.html</li>
	<li>extract_character_from_XString_by_positions.html</li>
	<li>extract_character_from_XString_by_ranges+2CXString-method.html</li>
	<li>extract_character_from_XString_by_ranges.html</li>
	<li>fasta.index.html</li>
	<li>fasta.seqlengths.html</li>
	<li>fastq.geometry.html</li>
	<li>fastq.seqlengths.html</li>
	<li>findPalindromes+2CDNAString-method.html</li>
	<li>findPalindromes+2CMaskedXString-method.html</li>
	<li>findPalindromes+2CRNAString-method.html</li>
	<li>findPalindromes+2CXString-method.html</li>
	<li>findPalindromes+2CXStringViews-method.html</li>
	<li>findPalindromes.html</li>
	<li>gaps+2CMaskedXString-method.html</li>
	<li>getGeneticCode.html</li>
	<li>getSeq.html</li>
	<li>get_seqtype_conversion_lookup.html</li>
	<li>gregexpr2.html</li>
	<li>hasAllFlinks+2CACtree2-method.html</li>
	<li>hasAllFlinks.html</li>
	<li>hasLetterAt.html</li>
	<li>hasOnlyBaseLetters+2CDNAString-method.html</li>
	<li>hasOnlyBaseLetters+2CDNAStringSet-method.html</li>
	<li>hasOnlyBaseLetters+2CMaskedXString-method.html</li>
	<li>hasOnlyBaseLetters+2CRNAString-method.html</li>
	<li>hasOnlyBaseLetters+2CRNAStringSet-method.html</li>
	<li>hasOnlyBaseLetters+2CXStringViews-method.html</li>
	<li>hasOnlyBaseLetters.html</li>
	<li>head+2CPDict3Parts-method.html</li>
	<li>head+2CTB_PDict-method.html</li>
	<li>indel+2CAlignedXStringSet0-method.html</li>
	<li>indel+2CPairwiseAlignments-method.html</li>
	<li>indel.html</li>
	<li>initialize+2CACtree2-method.html</li>
	<li>initialize+2CPreprocessedTB-method.html</li>
	<li>initialize+2CTwobit-method.html</li>
	<li>initialize+2CXStringCodec-method.html</li>
	<li>injectHardMask+2CMaskedXString-method.html</li>
	<li>injectHardMask+2CXStringViews-method.html</li>
	<li>injectHardMask.html</li>
	<li>insertion+2CInDel-method.html</li>
	<li>insertion+2CPairwiseAlignments-method.html</li>
	<li>insertion.html</li>
	<li>is.na+2CXStringSet-method.html</li>
	<li>isMatchingAt.html</li>
	<li>isMatchingEndingAt+2CXString-method.html</li>
	<li>isMatchingEndingAt+2CXStringSet-method.html</li>
	<li>isMatchingEndingAt+2Ccharacter-method.html</li>
	<li>isMatchingEndingAt.html</li>
	<li>isMatchingStartingAt+2CXString-method.html</li>
	<li>isMatchingStartingAt+2CXStringSet-method.html</li>
	<li>isMatchingStartingAt+2Ccharacter-method.html</li>
	<li>isMatchingStartingAt.html</li>
	<li>lcprefix+2CXString+2CXString-method.html</li>
	<li>lcprefix+2CXString+2Ccharacter-method.html</li>
	<li>lcprefix+2Ccharacter+2CXString-method.html</li>
	<li>lcprefix+2Ccharacter+2Ccharacter-method.html</li>
	<li>lcprefix.html</li>
	<li>lcsubstr+2CXString+2CXString-method.html</li>
	<li>lcsubstr+2CXString+2Ccharacter-method.html</li>
	<li>lcsubstr+2Ccharacter+2CXString-method.html</li>
	<li>lcsubstr+2Ccharacter+2Ccharacter-method.html</li>
	<li>lcsubstr.html</li>
	<li>lcsuffix+2CXString+2CXString-method.html</li>
	<li>lcsuffix+2CXString+2Ccharacter-method.html</li>
	<li>lcsuffix+2Ccharacter+2CXString-method.html</li>
	<li>lcsuffix+2Ccharacter+2Ccharacter-method.html</li>
	<li>lcsuffix.html</li>
	<li>length+2CMIndex-method.html</li>
	<li>length+2CMaskedXString-method.html</li>
	<li>length+2CPDict-method.html</li>
	<li>length+2CPDict3Parts-method.html</li>
	<li>length+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>length+2CPreprocessedTB-method.html</li>
	<li>length+2CSparseList-method.html</li>
	<li>letter+2CMaskedXString-method.html</li>
	<li>letter+2CXString-method.html</li>
	<li>letter+2CXStringViews-method.html</li>
	<li>letter+2Ccharacter-method.html</li>
	<li>letter.html</li>
	<li>letterFrequency+2CMaskedXString-method.html</li>
	<li>letterFrequency+2CXString-method.html</li>
	<li>letterFrequency+2CXStringSet-method.html</li>
	<li>letterFrequency+2CXStringViews-method.html</li>
	<li>letterFrequency.html</li>
	<li>letterFrequencyInSlidingView+2CXString-method.html</li>
	<li>letterFrequencyInSlidingView.html</li>
	<li>longestConsecutive.html</li>
	<li>lowlevel-matching.html</li>
	<li>make_XStringSet_from_strings+2CXStringSet-method.html</li>
	<li>make_XStringSet_from_strings.html</li>
	<li>make_XString_from_string+2CXString-method.html</li>
	<li>make_XString_from_string.html</li>
	<li>mask.html</li>
	<li>maskGaps+2CMultipleAlignment-method.html</li>
	<li>maskGaps.html</li>
	<li>maskMotif+2CMaskedXString+2CXString-method.html</li>
	<li>maskMotif+2CMaskedXString+2Ccharacter-method.html</li>
	<li>maskMotif+2CMultipleAlignment+2CANY-method.html</li>
	<li>maskMotif+2CXString+2CANY-method.html</li>
	<li>maskMotif.html</li>
	<li>maskeddim+2CMultipleAlignment-method.html</li>
	<li>maskeddim.html</li>
	<li>maskedncol+2CMultipleAlignment-method.html</li>
	<li>maskedncol.html</li>
	<li>maskednrow+2CMultipleAlignment-method.html</li>
	<li>maskednrow.html</li>
	<li>maskedratio+2CMaskedXString-method.html</li>
	<li>maskedratio+2CMultipleAlignment-method.html</li>
	<li>maskedwidth+2CMaskedXString-method.html</li>
	<li>masks+2CMaskedXString-method.html</li>
	<li>masks+2CXString-method.html</li>
	<li>masks+3C-+2CMaskedXString+2CMaskCollection-method.html</li>
	<li>masks+3C-+2CMaskedXString+2CNULL-method.html</li>
	<li>masks+3C-+2CXString+2CANY-method.html</li>
	<li>masks+3C-+2CXString+2CNULL-method.html</li>
	<li>masks+3C-.html</li>
	<li>masks.html</li>
	<li>match+2CANY+2CXStringSet-method.html</li>
	<li>match+2CXStringSet+2CANY-method.html</li>
	<li>match+2CXStringSet+2CXStringSet-method.html</li>
	<li>match-utils.html</li>
	<li>matchLRPatterns+2CMaskedXString-method.html</li>
	<li>matchLRPatterns+2CXString-method.html</li>
	<li>matchLRPatterns+2CXStringViews-method.html</li>
	<li>matchLRPatterns.html</li>
	<li>matchPDict+2CMaskedXString-method.html</li>
	<li>matchPDict+2CXString-method.html</li>
	<li>matchPDict+2CXStringSet-method.html</li>
	<li>matchPDict+2CXStringViews-method.html</li>
	<li>matchPDict-exact.html</li>
	<li>matchPDict-inexact.html</li>
	<li>matchPDict.html</li>
	<li>matchPWM+2CDNAString-method.html</li>
	<li>matchPWM+2CMaskedDNAString-method.html</li>
	<li>matchPWM+2CXStringViews-method.html</li>
	<li>matchPWM+2Ccharacter-method.html</li>
	<li>matchPWM.html</li>
	<li>matchPattern+2CMaskedXString-method.html</li>
	<li>matchPattern+2CXString-method.html</li>
	<li>matchPattern+2CXStringSet-method.html</li>
	<li>matchPattern+2CXStringViews-method.html</li>
	<li>matchPattern+2Ccharacter-method.html</li>
	<li>matchPattern.html</li>
	<li>matchProbePair+2CDNAString-method.html</li>
	<li>matchProbePair+2CMaskedDNAString-method.html</li>
	<li>matchProbePair+2CXStringViews-method.html</li>
	<li>matchProbePair.html</li>
	<li>matchprobes.html</li>
	<li>maxScore+2CANY-method.html</li>
	<li>maxScore.html</li>
	<li>maxWeights+2Cmatrix-method.html</li>
	<li>maxWeights.html</li>
	<li>mergeIUPACLetters.html</li>
	<li>minScore+2CANY-method.html</li>
	<li>minScore.html</li>
	<li>minWeights+2Cmatrix-method.html</li>
	<li>minWeights.html</li>
	<li>misc.html</li>
	<li>mismatch+2CANY+2CXStringViews-method.html</li>
	<li>mismatch+2CAlignedXStringSet0+2Cmissing-method.html</li>
	<li>mismatch.html</li>
	<li>mismatchSummary+2CAlignedXStringSet0-method.html</li>
	<li>mismatchSummary+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>mismatchSummary+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>mismatchSummary+2CQualityAlignedXStringSet-method.html</li>
	<li>mismatchSummary.html</li>
	<li>mismatchTable+2CAlignedXStringSet0-method.html</li>
	<li>mismatchTable+2CPairwiseAlignments-method.html</li>
	<li>mismatchTable+2CQualityAlignedXStringSet-method.html</li>
	<li>mismatchTable.html</li>
	<li>mkAllStrings.html</li>
	<li>names+2CMIndex-method.html</li>
	<li>names+2CPDict-method.html</li>
	<li>names+3C-+2CMIndex-method.html</li>
	<li>names+3C-+2CPDict-method.html</li>
	<li>narrow+2CQualityScaledXStringSet-method.html</li>
	<li>narrow+2Ccharacter-method.html</li>
	<li>nchar+2CAlignedXStringSet0-method.html</li>
	<li>nchar+2CMaskedXString-method.html</li>
	<li>nchar+2CMultipleAlignment-method.html</li>
	<li>nchar+2CPairwiseAlignments-method.html</li>
	<li>nchar+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>nchar+2CXString-method.html</li>
	<li>nchar+2CXStringSet-method.html</li>
	<li>nchar+2CXStringSetList-method.html</li>
	<li>nchar+2CXStringViews-method.html</li>
	<li>ncol+2CMultipleAlignment-method.html</li>
	<li>nedit+2CPairwiseAlignments-method.html</li>
	<li>nedit+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>nedit.html</li>
	<li>neditAt.html</li>
	<li>neditEndingAt+2CXString-method.html</li>
	<li>neditEndingAt+2CXStringSet-method.html</li>
	<li>neditEndingAt+2Ccharacter-method.html</li>
	<li>neditEndingAt.html</li>
	<li>neditStartingAt+2CXString-method.html</li>
	<li>neditStartingAt+2CXStringSet-method.html</li>
	<li>neditStartingAt+2Ccharacter-method.html</li>
	<li>neditStartingAt.html</li>
	<li>needwunsQS+2CXString+2CXString-method.html</li>
	<li>needwunsQS+2CXString+2Ccharacter-method.html</li>
	<li>needwunsQS+2Ccharacter+2CXString-method.html</li>
	<li>needwunsQS+2Ccharacter+2Ccharacter-method.html</li>
	<li>needwunsQS.html</li>
	<li>nindel+2CAlignedXStringSet0-method.html</li>
	<li>nindel+2CPairwiseAlignments-method.html</li>
	<li>nindel+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>nindel.html</li>
	<li>nmatch+2CANY+2CXStringViews-method.html</li>
	<li>nmatch+2CPairwiseAlignments+2Cmissing-method.html</li>
	<li>nmatch+2CPairwiseAlignmentsSingleSubjectSummary+2Cmissing-method.html</li>
	<li>nmatch.html</li>
	<li>nmismatch+2CANY+2CXStringViews-method.html</li>
	<li>nmismatch+2CAlignedXStringSet0+2Cmissing-method.html</li>
	<li>nmismatch+2CPairwiseAlignments+2Cmissing-method.html</li>
	<li>nmismatch+2CPairwiseAlignmentsSingleSubjectSummary+2Cmissing-method.html</li>
	<li>nmismatch.html</li>
	<li>nnodes+2CACtree2-method.html</li>
	<li>nnodes.html</li>
	<li>nrow+2CMultipleAlignment-method.html</li>
	<li>nucleotideFrequency.html</li>
	<li>nucleotideFrequencyAt+2CXStringSet-method.html</li>
	<li>nucleotideFrequencyAt+2CXStringViews-method.html</li>
	<li>nucleotideFrequencyAt.html</li>
	<li>nucleotideSubstitutionMatrix.html</li>
	<li>oligonucleotideFrequency+2CMaskedXString-method.html</li>
	<li>oligonucleotideFrequency+2CXString-method.html</li>
	<li>oligonucleotideFrequency+2CXStringSet-method.html</li>
	<li>oligonucleotideFrequency+2CXStringViews-method.html</li>
	<li>oligonucleotideFrequency.html</li>
	<li>oligonucleotideTransitions.html</li>
	<li>padAndClip.html</li>
	<li>pairwiseAlignment+2CANY+2CANY-method.html</li>
	<li>pairwiseAlignment+2CANY+2CQualityScaledXStringSet-method.html</li>
	<li>pairwiseAlignment+2CQualityScaledXStringSet+2CANY-method.html</li>
	<li>pairwiseAlignment+2CQualityScaledXStringSet+2CQualityScaledXStringSet-method.html</li>
	<li>pairwiseAlignment.html</li>
	<li>palindromeArmLength+2CDNAString-method.html</li>
	<li>palindromeArmLength+2CRNAString-method.html</li>
	<li>palindromeArmLength+2CXString-method.html</li>
	<li>palindromeArmLength+2CXStringSet-method.html</li>
	<li>palindromeArmLength+2CXStringViews-method.html</li>
	<li>palindromeArmLength.html</li>
	<li>palindromeLeftArm+2CXString-method.html</li>
	<li>palindromeLeftArm+2CXStringViews-method.html</li>
	<li>palindromeLeftArm.html</li>
	<li>palindromeRightArm+2CXString-method.html</li>
	<li>palindromeRightArm+2CXStringViews-method.html</li>
	<li>palindromeRightArm.html</li>
	<li>parallelVectorNames+2CAlignedXStringSet0-method.html</li>
	<li>parallel_slot_names+2CAlignedXStringSet0-method.html</li>
	<li>parallel_slot_names+2CPairwiseAlignments-method.html</li>
	<li>parallel_slot_names+2CQualityScaledXStringSet-method.html</li>
	<li>paths.rds</li>
	<li>pattern+2CPairwiseAlignments-method.html</li>
	<li>pattern+2CXStringPartialMatches-method.html</li>
	<li>pattern.html</li>
	<li>patternFrequency+2CPDict-method.html</li>
	<li>patternFrequency.html</li>
	<li>pcompare+2CANY+2CXStringSet-method.html</li>
	<li>pcompare+2CXStringSet+2CANY-method.html</li>
	<li>pcompare+2CXStringSet+2CXStringSet-method.html</li>
	<li>phiX174Phage.html</li>
	<li>pid+2CPairwiseAlignments-method.html</li>
	<li>pid.html</li>
	<li>pmatchPattern+2CXString-method.html</li>
	<li>pmatchPattern+2CXStringViews-method.html</li>
	<li>pmatchPattern+2Ccharacter-method.html</li>
	<li>pmatchPattern.html</li>
	<li>print.needwunsQS.html</li>
	<li>quPhiX174.html</li>
	<li>quality+2CQualityScaledXStringSet-method.html</li>
	<li>quality.html</li>
	<li>qualitySubstitutionMatrices.html</li>
	<li>ranges+2CAlignedXStringSet0-method.html</li>
	<li>readAAMultipleAlignment.html</li>
	<li>readAAStringSet.html</li>
	<li>readBStringSet.html</li>
	<li>readDNAMultipleAlignment.html</li>
	<li>readDNAStringSet.html</li>
	<li>readQualityScaledDNAStringSet.html</li>
	<li>readRNAMultipleAlignment.html</li>
	<li>readRNAStringSet.html</li>
	<li>relistToClass+2CXStringSet-method.html</li>
	<li>replaceAmbiguities.html</li>
	<li>replaceAt+2CXString-method.html</li>
	<li>replaceAt+2CXStringSet-method.html</li>
	<li>replaceAt.html</li>
	<li>replaceLetterAt+2CDNAString-method.html</li>
	<li>replaceLetterAt+2CDNAStringSet-method.html</li>
	<li>replaceLetterAt.html</li>
	<li>reverse+2CMaskedXString-method.html</li>
	<li>reverse+2CQualityScaledXStringSet-method.html</li>
	<li>reverseComplement+2CDNAString-method.html</li>
	<li>reverseComplement+2CDNAStringSet-method.html</li>
	<li>reverseComplement+2CMaskedDNAString-method.html</li>
	<li>reverseComplement+2CMaskedRNAString-method.html</li>
	<li>reverseComplement+2CQualityScaledDNAStringSet-method.html</li>
	<li>reverseComplement+2CQualityScaledRNAStringSet-method.html</li>
	<li>reverseComplement+2CRNAString-method.html</li>
	<li>reverseComplement+2CRNAStringSet-method.html</li>
	<li>reverseComplement+2CXStringViews-method.html</li>
	<li>reverseComplement+2Cmatrix-method.html</li>
	<li>reverseComplement.html</li>
	<li>rowmask+2CMultipleAlignment-method.html</li>
	<li>rowmask+3C-+2CMultipleAlignment+2CANY-method.html</li>
	<li>rowmask+3C-+2CMultipleAlignment+2CNULL-method.html</li>
	<li>rowmask+3C-.html</li>
	<li>rowmask.html</li>
	<li>rownames+2CMultipleAlignment-method.html</li>
	<li>rownames+3C-+2CMultipleAlignment-method.html</li>
	<li>saveXStringSet.html</li>
	<li>score+2CPairwiseAlignments-method.html</li>
	<li>score+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>seqinfo+2CDNAStringSet-method.html</li>
	<li>seqinfo+3C-+2CDNAStringSet-method.html</li>
	<li>seqinfo.html</li>
	<li>seqtype+2CAAString-method.html</li>
	<li>seqtype+2CAlignedXStringSet0-method.html</li>
	<li>seqtype+2CBString-method.html</li>
	<li>seqtype+2CDNAString-method.html</li>
	<li>seqtype+2CMaskedXString-method.html</li>
	<li>seqtype+2CMultipleAlignment-method.html</li>
	<li>seqtype+2CPairwiseAlignments-method.html</li>
	<li>seqtype+2CRNAString-method.html</li>
	<li>seqtype+2CXStringSet-method.html</li>
	<li>seqtype+2CXStringSetList-method.html</li>
	<li>seqtype+2CXStringViews-method.html</li>
	<li>seqtype+3C-+2CMaskedXString-method.html</li>
	<li>seqtype+3C-+2CXString-method.html</li>
	<li>seqtype+3C-+2CXStringSet-method.html</li>
	<li>seqtype+3C-+2CXStringSetList-method.html</li>
	<li>seqtype+3C-+2CXStringViews-method.html</li>
	<li>seqtype+3C-.html</li>
	<li>seqtype.html</li>
	<li>show+2CACtree2-method.html</li>
	<li>show+2CAlignedXStringSet0-method.html</li>
	<li>show+2CMIndex-method.html</li>
	<li>show+2CMTB_PDict-method.html</li>
	<li>show+2CMaskedXString-method.html</li>
	<li>show+2CMultipleAlignment-method.html</li>
	<li>show+2CPairwiseAlignments-method.html</li>
	<li>show+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>show+2CQualityScaledXStringSet-method.html</li>
	<li>show+2CTB_PDict-method.html</li>
	<li>show+2CTwobit-method.html</li>
	<li>show+2CXString-method.html</li>
	<li>show+2CXStringPartialMatches-method.html</li>
	<li>show+2CXStringSet-method.html</li>
	<li>show+2CXStringSetList-method.html</li>
	<li>show+2CXStringViews-method.html</li>
	<li>showAsCell+2CXString-method.html</li>
	<li>showAsCell+2CXStringSet-method.html</li>
	<li>showAsCell+2CXStringSetList-method.html</li>
	<li>srPhiX174.html</li>
	<li>stackStrings.html</li>
	<li>start+2CAlignedXStringSet0-method.html</li>
	<li>startIndex+2CByPos_MIndex-method.html</li>
	<li>startIndex.html</li>
	<li>stringDist+2CQualityScaledXStringSet-method.html</li>
	<li>stringDist+2CXStringSet-method.html</li>
	<li>stringDist+2Ccharacter-method.html</li>
	<li>stringDist.html</li>
	<li>subject+2CPairwiseAlignments-method.html</li>
	<li>subpatterns+2CXStringPartialMatches-method.html</li>
	<li>subpatterns.html</li>
	<li>subseq+2CMaskedXString-method.html</li>
	<li>subseq+2Ccharacter-method.html</li>
	<li>subseq+3C-+2CXStringSet-method.html</li>
	<li>subseq+3C-+2Ccharacter-method.html</li>
	<li>substitution.matrices.html</li>
	<li>substitution_matrices.html</li>
	<li>substr+2CXString-method.html</li>
	<li>substring+2CXString-method.html</li>
	<li>summary+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>tail+2CPDict3Parts-method.html</li>
	<li>tail+2CTB_PDict-method.html</li>
	<li>tb+2CPDict3Parts-method.html</li>
	<li>tb+2CPreprocessedTB-method.html</li>
	<li>tb+2CTB_PDict-method.html</li>
	<li>tb.html</li>
	<li>tb.width+2CPDict3Parts-method.html</li>
	<li>tb.width+2CPreprocessedTB-method.html</li>
	<li>tb.width+2CTB_PDict-method.html</li>
	<li>tb.width.html</li>
	<li>threebands+2Ccharacter-method.html</li>
	<li>toComplex+2CDNAString-method.html</li>
	<li>toComplex.html</li>
	<li>toString+2CAlignedXStringSet0-method.html</li>
	<li>toString+2CMaskedXString-method.html</li>
	<li>toString+2CPairwiseAlignmentsSingleSubject-method.html</li>
	<li>toString+2CXString-method.html</li>
	<li>toString+2CXStringSet-method.html</li>
	<li>toString+2CXStringViews-method.html</li>
	<li>translate+2CDNAString-method.html</li>
	<li>translate+2CDNAStringSet-method.html</li>
	<li>translate+2CMaskedDNAString-method.html</li>
	<li>translate+2CMaskedRNAString-method.html</li>
	<li>translate+2CRNAString-method.html</li>
	<li>translate+2CRNAStringSet-method.html</li>
	<li>translate.html</li>
	<li>trimLRPatterns+2CXString-method.html</li>
	<li>trimLRPatterns+2CXStringSet-method.html</li>
	<li>trimLRPatterns+2Ccharacter-method.html</li>
	<li>trimLRPatterns.html</li>
	<li>trinucleotideFrequency.html</li>
	<li>type+2CPairwiseAlignments-method.html</li>
	<li>type+2CPairwiseAlignmentsSingleSubjectSummary-method.html</li>
	<li>type.html</li>
	<li>unaligned+2CAlignedXStringSet0-method.html</li>
	<li>unaligned.html</li>
	<li>uniqueLetters+2CMaskedXString-method.html</li>
	<li>uniqueLetters+2CXString-method.html</li>
	<li>uniqueLetters+2CXStringSet-method.html</li>
	<li>uniqueLetters+2CXStringViews-method.html</li>
	<li>uniqueLetters.html</li>
	<li>unitScale.html</li>
	<li>unlist+2CMIndex-method.html</li>
	<li>unlist+2CXStringSet-method.html</li>
	<li>unmasked+2CMaskedXString-method.html</li>
	<li>unmasked+2CMultipleAlignment-method.html</li>
	<li>unmasked+2CXString-method.html</li>
	<li>unmasked.html</li>
	<li>updateObject+2CXString-method.html</li>
	<li>updateObject+2CXStringSet-method.html</li>
	<li>vcountPDict+2CMaskedXString-method.html</li>
	<li>vcountPDict+2CXString-method.html</li>
	<li>vcountPDict+2CXStringSet-method.html</li>
	<li>vcountPDict+2CXStringViews-method.html</li>
	<li>vcountPDict.html</li>
	<li>vcountPattern+2CMaskedXString-method.html</li>
	<li>vcountPattern+2CXString-method.html</li>
	<li>vcountPattern+2CXStringSet-method.html</li>
	<li>vcountPattern+2CXStringViews-method.html</li>
	<li>vcountPattern+2Ccharacter-method.html</li>
	<li>vcountPattern.html</li>
	<li>vmatchPDict+2CANY-method.html</li>
	<li>vmatchPDict+2CMaskedXString-method.html</li>
	<li>vmatchPDict+2CXString-method.html</li>
	<li>vmatchPDict.html</li>
	<li>vmatchPattern+2CMaskedXString-method.html</li>
	<li>vmatchPattern+2CXString-method.html</li>
	<li>vmatchPattern+2CXStringSet-method.html</li>
	<li>vmatchPattern+2CXStringViews-method.html</li>
	<li>vmatchPattern+2Ccharacter-method.html</li>
	<li>vmatchPattern.html</li>
	<li>vwhichPDict+2CMaskedXString-method.html</li>
	<li>vwhichPDict+2CXString-method.html</li>
	<li>vwhichPDict+2CXStringSet-method.html</li>
	<li>vwhichPDict+2CXStringViews-method.html</li>
	<li>vwhichPDict.html</li>
	<li>which.isMatchingAt.html</li>
	<li>which.isMatchingEndingAt+2CXString-method.html</li>
	<li>which.isMatchingEndingAt+2CXStringSet-method.html</li>
	<li>which.isMatchingEndingAt+2Ccharacter-method.html</li>
	<li>which.isMatchingEndingAt.html</li>
	<li>which.isMatchingStartingAt+2CXString-method.html</li>
	<li>which.isMatchingStartingAt+2CXStringSet-method.html</li>
	<li>which.isMatchingStartingAt+2Ccharacter-method.html</li>
	<li>which.isMatchingStartingAt.html</li>
	<li>whichPDict+2CMaskedXString-method.html</li>
	<li>whichPDict+2CXString-method.html</li>
	<li>whichPDict+2CXStringSet-method.html</li>
	<li>whichPDict+2CXStringViews-method.html</li>
	<li>whichPDict.html</li>
	<li>width+2CAlignedXStringSet0-method.html</li>
	<li>width+2CPDict-method.html</li>
	<li>width+2CPDict3Parts-method.html</li>
	<li>width+2CPreprocessedTB-method.html</li>
	<li>width+2Ccharacter-method.html</li>
	<li>width0+2CMIndex-method.html</li>
	<li>width0.html</li>
	<li>windows+2CQualityScaledXStringSet-method.html</li>
	<li>windows+2Ccharacter-method.html</li>
	<li>write.phylip.html</li>
	<li>writePairwiseAlignments.html</li>
	<li>writeQualityScaledXStringSet.html</li>
	<li>writeXStringSet.html</li>
	<li>wtPhiX174.html</li>
	<li>xscat.html</li>
	<li>xscodes+2CANY-method.html</li>
	<li>xscodes.html</li>
	<li>yeastSEQCHR1.html</li>
	</ul></li>
	<li>html/<ul>
	<li>00Index.html</li>
	<li>AAString-class.html</li>
	<li>AMINO_ACID_CODE.html</li>
	<li>AlignedXStringSet-class.html</li>
	<li>Biostrings-internals.html</li>
	<li>DNAString-class.html</li>
	<li>GENETIC_CODE.html</li>
	<li>HNF4alpha.html</li>
	<li>IUPAC_CODE_MAP.html</li>
	<li>InDel-class.html</li>
	<li>MIndex-class.html</li>
	<li>MaskedXString-class.html</li>
	<li>MultipleAlignment-class.html</li>
	<li>PDict-class.html</li>
	<li>PairwiseAlignments-class.html</li>
	<li>PairwiseAlignments-io.html</li>
	<li>QualityScaledXStringSet-class.html</li>
	<li>R.css</li>
	<li>RNAString-class.html</li>
	<li>XString-class.html</li>
	<li>XStringPartialMatches-class.html</li>
	<li>XStringQuality-class.html</li>
	<li>XStringSet-class.html</li>
	<li>XStringSet-comparison.html</li>
	<li>XStringSet-io.html</li>
	<li>XStringSetList-class.html</li>
	<li>XStringViews-class.html</li>
	<li>align-utils.html</li>
	<li>chartr.html</li>
	<li>detail.html</li>
	<li>dinucleotideFrequencyTest.html</li>
	<li>findPalindromes.html</li>
	<li>getSeq.html</li>
	<li>gregexpr2.html</li>
	<li>injectHardMask.html</li>
	<li>letter.html</li>
	<li>letterFrequency.html</li>
	<li>longestConsecutive.html</li>
	<li>lowlevel-matching.html</li>
	<li>maskMotif.html</li>
	<li>match-utils.html</li>
	<li>matchLRPatterns.html</li>
	<li>matchPDict-exact.html</li>
	<li>matchPDict-inexact.html</li>
	<li>matchPWM.html</li>
	<li>matchPattern.html</li>
	<li>matchProbePair.html</li>
	<li>matchprobes.html</li>
	<li>misc.html</li>
	<li>needwunsQS.html</li>
	<li>nucleotideFrequency.html</li>
	<li>padAndClip.html</li>
	<li>pairwiseAlignment.html</li>
	<li>phiX174Phage.html</li>
	<li>pid.html</li>
	<li>pmatchPattern.html</li>
	<li>replaceAt.html</li>
	<li>replaceLetterAt.html</li>
	<li>reverseComplement.html</li>
	<li>seqinfo.html</li>
	<li>stringDist.html</li>
	<li>substitution_matrices.html</li>
	<li>toComplex.html</li>
	<li>translate.html</li>
	<li>trimLRPatterns.html</li>
	<li>xscat.html</li>
	<li>yeastSEQCHR1.html</li>
	</ul></li>
	<li>libs/<ul>
	<li>Biostrings.so</li>
	</ul></li>
	<li>unitTests/<ul>
	<li>test_MultipleAlignment.R</li>
	<li>test_XString.R</li>
	<li>test_XStringSet-io.R</li>
	<li>test_XStringSet.R</li>
	<li>test_XStringSetList.R</li>
	<li>test_findPalindromes.R</li>
	<li>test_matchDNAPattern.R.FIXME</li>
	<li>test_matchPDict.R</li>
	<li>test_pairwiseAlignment.R</li>
	<li>test_seqinfo.R</li>
	<li>test_twoWayAlphabetFrequency.R</li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
	</ul></li>
</div>

  <div class="text-muted text-center">
    <br>
    Sources on <a href="https://pagure.io/fedora-packages-static">Pagure</a>
  </div>
</div>